<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head><title>R: Annotation for microarrays</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="stylesheet" type="text/css" href="R.css" />
</head><body>
<h1> Annotation for microarrays
<img class="toplogo" src="../../../doc/html/Rlogo.svg" alt="[R logo]" />
</h1>
<hr/>
<div style="text-align: center;">
<a href="../../../doc/html/packages.html"><img class="arrow" src="../../../doc/html/left.jpg" alt="[Up]" /></a>
<a href="../../../doc/html/index.html"><img class="arrow" src="../../../doc/html/up.jpg" alt="[Top]" /></a>
</div><h2>Documentation for package &lsquo;annotate&rsquo; version 1.60.1</h2>

<ul><li><a href="../DESCRIPTION">DESCRIPTION file</a>.</li>
<li><a href="../doc/index.html">User guides, package vignettes and other documentation.</a></li>
<li><a href="../NEWS">Package NEWS</a>.</li>
</ul>

<h2>Help Pages</h2>


<p style="text-align: center;">
<a href="#A">A</a>
<a href="#B">B</a>
<a href="#C">C</a>
<a href="#D">D</a>
<a href="#E">E</a>
<a href="#F">F</a>
<a href="#G">G</a>
<a href="#H">H</a>
<a href="#I">I</a>
<a href="#J">J</a>
<a href="#K">K</a>
<a href="#L">L</a>
<a href="#M">M</a>
<a href="#N">N</a>
<a href="#O">O</a>
<a href="#P">P</a>
<a href="#R">R</a>
<a href="#S">S</a>
<a href="#T">T</a>
<a href="#U">U</a>
<a href="#W">W</a>
</p>


<h2><a name="A">-- A --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="pubMedAbst-class.html">abstText</a></td>
<td>Class pubMedAbst, a class to handle PubMed abstracts, and methods for processing them.</td></tr>
<tr><td style="width: 25%;"><a href="pubMedAbst-class.html">abstText-method</a></td>
<td>Class pubMedAbst, a class to handle PubMed abstracts, and methods for processing them.</td></tr>
<tr><td style="width: 25%;"><a href="LL2homology.html">ACC2homology</a></td>
<td>DEPRECATED Functions that find the homology data for a given set of LocusLink ids or HomoloGeneIDs</td></tr>
<tr><td style="width: 25%;"><a href="accessionToUID.html">accessionToUID</a></td>
<td>A function to convert accession values to NCBI UIDs.</td></tr>
<tr><td style="width: 25%;"><a href="ACCNUMStats.html">ACCNUMStats</a></td>
<td>Provides statistics on the types of ids used for the ACCNUM environment of a given data package</td></tr>
<tr><td style="width: 25%;"><a href="isValidkey.html">allValidKeys</a></td>
<td>Get or verify valid IDs for a package or OrgDb object.</td></tr>
<tr><td style="width: 25%;"><a href="isValidkey.html">allValidKeys-method</a></td>
<td>Get or verify valid IDs for a package or OrgDb object.</td></tr>
<tr><td style="width: 25%;"><a href="annotate-defunct.html">annotate-defunct</a></td>
<td>Defunct Functions in Package 'annotate'</td></tr>
<tr><td style="width: 25%;"><a href="annPkgName.html">annPkgName</a></td>
<td>Get annotation package name from chip name</td></tr>
<tr><td style="width: 25%;"><a href="aqListGOIDs.html">aqListGOIDs</a></td>
<td>List GO Identifiers by GO Ontology</td></tr>
<tr><td style="width: 25%;"><a href="pubMedAbst-class.html">articleTitle</a></td>
<td>Class pubMedAbst, a class to handle PubMed abstracts, and methods for processing them.</td></tr>
<tr><td style="width: 25%;"><a href="pubMedAbst-class.html">articleTitle-method</a></td>
<td>Class pubMedAbst, a class to handle PubMed abstracts, and methods for processing them.</td></tr>
<tr><td style="width: 25%;"><a href="pubMedAbst-class.html">authors</a></td>
<td>Class pubMedAbst, a class to handle PubMed abstracts, and methods for processing them.</td></tr>
<tr><td style="width: 25%;"><a href="pubMedAbst-class.html">authors-method</a></td>
<td>Class pubMedAbst, a class to handle PubMed abstracts, and methods for processing them.</td></tr>
</table>

<h2><a name="B">-- B --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="blastSequences.html">blastSequences</a></td>
<td>Run a blast query to NCBI for either a string or an entrez gene ID and then return a series of MultipleAlignment objects.</td></tr>
<tr><td style="width: 25%;"><a href="buildChromLocation.html">buildChromLocation</a></td>
<td>A function to generate an instantiation of a chromLocation class</td></tr>
<tr><td style="width: 25%;"><a href="buildPubMedAbst.html">buildPubMedAbst</a></td>
<td>A function to generate an instantiation of a pubMedAbst class</td></tr>
</table>

<h2><a name="C">-- C --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="findNeighbors.html">checkArgs</a></td>
<td>A function to locate neighboring genes within a defined range around a target gene represented by a Entrez Gene ID</td></tr>
<tr><td style="width: 25%;"><a href="chrCats.html">chrCats</a></td>
<td>Returns a list of chromosome locations from a MAP environment</td></tr>
<tr><td style="width: 25%;"><a href="chromLocation-class.html">chromInfo</a></td>
<td>Class chromLocation, a class for describing genes and their chromosome mappings.</td></tr>
<tr><td style="width: 25%;"><a href="chromLocation-class.html">chromInfo-method</a></td>
<td>Class chromLocation, a class for describing genes and their chromosome mappings.</td></tr>
<tr><td style="width: 25%;"><a href="chromLocation-class.html">chromLengths</a></td>
<td>Class chromLocation, a class for describing genes and their chromosome mappings.</td></tr>
<tr><td style="width: 25%;"><a href="chromLocation-class.html">chromLengths-method</a></td>
<td>Class chromLocation, a class for describing genes and their chromosome mappings.</td></tr>
<tr><td style="width: 25%;"><a href="chromLocation-class.html">chromLocation</a></td>
<td>Class chromLocation, a class for describing genes and their chromosome mappings.</td></tr>
<tr><td style="width: 25%;"><a href="chromLocation-class.html">chromLocation-class</a></td>
<td>Class chromLocation, a class for describing genes and their chromosome mappings.</td></tr>
<tr><td style="width: 25%;"><a href="chromLocation-class.html">chromLocs</a></td>
<td>Class chromLocation, a class for describing genes and their chromosome mappings.</td></tr>
<tr><td style="width: 25%;"><a href="chromLocation-class.html">chromLocs-method</a></td>
<td>Class chromLocation, a class for describing genes and their chromosome mappings.</td></tr>
<tr><td style="width: 25%;"><a href="chromLocation-class.html">chromNames</a></td>
<td>Class chromLocation, a class for describing genes and their chromosome mappings.</td></tr>
<tr><td style="width: 25%;"><a href="chromLocation-class.html">chromNames-method</a></td>
<td>Class chromLocation, a class for describing genes and their chromosome mappings.</td></tr>
<tr><td style="width: 25%;"><a href="setRepository.html">clearRepository</a></td>
<td>Functions to add arbitrary repositories</td></tr>
<tr><td style="width: 25%;"><a href="compatibleVersions.html">compatibleVersions</a></td>
<td>function to check to see if the packages represented by the names passed have the same version number</td></tr>
<tr><td style="width: 25%;"><a href="chrCats.html">createLLChrCats</a></td>
<td>Returns a list of chromosome locations from a MAP environment</td></tr>
<tr><td style="width: 25%;"><a href="chrCats.html">createMAPIncMat</a></td>
<td>Returns a list of chromosome locations from a MAP environment</td></tr>
</table>

<h2><a name="D">-- D --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="chromLocation-class.html">dataSource</a></td>
<td>Class chromLocation, a class for describing genes and their chromosome mappings.</td></tr>
<tr><td style="width: 25%;"><a href="chromLocation-class.html">dataSource-method</a></td>
<td>Class chromLocation, a class for describing genes and their chromosome mappings.</td></tr>
<tr><td style="width: 25%;"><a href="dropECode.html">dropECode</a></td>
<td>Drop GO labels for specified Evidence Codes</td></tr>
</table>

<h2><a name="E">-- E --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="entrezGeneByID.html">entrezGeneByID</a></td>
<td>Create a Query String for an Entrez Gene Identifier</td></tr>
<tr><td style="width: 25%;"><a href="entrezGeneQuery.html">entrezGeneQuery</a></td>
<td>Create a Query String for Entrez Genes</td></tr>
</table>

<h2><a name="F">-- F --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="HTMLPage-class.html">fileName</a></td>
<td>Classes to represent HTML pages</td></tr>
<tr><td style="width: 25%;"><a href="HTMLPage-class.html">fileName-method</a></td>
<td>Classes to represent HTML pages</td></tr>
<tr><td style="width: 25%;"><a href="filterGOByOntology.html">filterGOByOntology</a></td>
<td>Filter GO terms by a specified GO ontology</td></tr>
<tr><td style="width: 25%;"><a href="findNeighbors.html">findChr4LL</a></td>
<td>A function to locate neighboring genes within a defined range around a target gene represented by a Entrez Gene ID</td></tr>
<tr><td style="width: 25%;"><a href="findNeighbors.html">findNeighbors</a></td>
<td>A function to locate neighboring genes within a defined range around a target gene represented by a Entrez Gene ID</td></tr>
<tr><td style="width: 25%;"><a href="HTMLPage-class.html">FramedHTMLPage</a></td>
<td>Classes to represent HTML pages</td></tr>
<tr><td style="width: 25%;"><a href="HTMLPage-class.html">FramedHTMLPage-class</a></td>
<td>Classes to represent HTML pages</td></tr>
<tr><td style="width: 25%;"><a href="HTMLPage-class.html">FramedHTMLPage-method</a></td>
<td>Classes to represent HTML pages</td></tr>
</table>

<h2><a name="G">-- G --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="genbank.html">genbank</a></td>
<td>A function to open the browser to Genbank with the selected gene.</td></tr>
<tr><td style="width: 25%;"><a href="annotate-defunct.html">genelocator</a></td>
<td>Defunct Functions in Package 'annotate'</td></tr>
<tr><td style="width: 25%;"><a href="chromLocation-class.html">geneSymbols</a></td>
<td>Class chromLocation, a class for describing genes and their chromosome mappings.</td></tr>
<tr><td style="width: 25%;"><a href="chromLocation-class.html">geneSymbols-method</a></td>
<td>Class chromLocation, a class for describing genes and their chromosome mappings.</td></tr>
<tr><td style="width: 25%;"><a href="getAnnMap.html">getAnnMap</a></td>
<td>Get annotation map</td></tr>
<tr><td style="width: 25%;"><a href="findNeighbors.html">getBoundary</a></td>
<td>A function to locate neighboring genes within a defined range around a target gene represented by a Entrez Gene ID</td></tr>
<tr><td style="width: 25%;"><a href="getTDRows.html">getCells</a></td>
<td>Functions to create hypertext links that can be placed in a table cell of a HTML file</td></tr>
<tr><td style="width: 25%;"><a href="getSYMBOL.html">getEG</a></td>
<td>Functions to deal with Data Packages</td></tr>
<tr><td style="width: 25%;"><a href="getEvidence.html">getEvidence</a></td>
<td>Get the Evidence codes for a set of GO terms.</td></tr>
<tr><td style="width: 25%;"><a href="getSeq4Acc.html">getGI</a></td>
<td>Queries the NCBI database to obtain the sequence for a given GenBank Accession number</td></tr>
<tr><td style="width: 25%;"><a href="getSYMBOL.html">getGO</a></td>
<td>Functions to deal with Data Packages</td></tr>
<tr><td style="width: 25%;"><a href="getGOTerm.html">getGOChildren</a></td>
<td>Functions to Access GO data.</td></tr>
<tr><td style="width: 25%;"><a href="getSYMBOL.html">getGOdesc</a></td>
<td>Functions to deal with Data Packages</td></tr>
<tr><td style="width: 25%;"><a href="getGOTerm.html">getGOOntology</a></td>
<td>Functions to Access GO data.</td></tr>
<tr><td style="width: 25%;"><a href="getGOTerm.html">getGOParents</a></td>
<td>Functions to Access GO data.</td></tr>
<tr><td style="width: 25%;"><a href="getGOTerm.html">getGOTerm</a></td>
<td>Functions to Access GO data.</td></tr>
<tr><td style="width: 25%;"><a href="readGEOAnn.html">getGPLNames</a></td>
<td>Function to extract data from the GEO web site</td></tr>
<tr><td style="width: 25%;"><a href="getSYMBOL.html">getLL</a></td>
<td>Functions to deal with Data Packages</td></tr>
<tr><td style="width: 25%;"><a href="getOntology.html">getOntology</a></td>
<td>Get GO terms for a specified ontology</td></tr>
<tr><td style="width: 25%;"><a href="mapOrgs.html">getOrgNameNCode</a></td>
<td>Functions to map to organism IDs used by NCBI homology.</td></tr>
<tr><td style="width: 25%;"><a href="getSYMBOL.html">getPMID</a></td>
<td>Functions to deal with Data Packages</td></tr>
<tr><td style="width: 25%;"><a href="getPMInfo.html">getPMInfo</a></td>
<td>extract publication details and abstract from annotate::pubmed function output</td></tr>
<tr><td style="width: 25%;"><a href="getTDRows.html">getQuery4Affy</a></td>
<td>Functions to create hypertext links that can be placed in a table cell of a HTML file</td></tr>
<tr><td style="width: 25%;"><a href="getTDRows.html">getQuery4EN</a></td>
<td>Functions to create hypertext links that can be placed in a table cell of a HTML file</td></tr>
<tr><td style="width: 25%;"><a href="getTDRows.html">getQuery4ENSEMBL</a></td>
<td>Functions to create hypertext links that can be placed in a table cell of a HTML file</td></tr>
<tr><td style="width: 25%;"><a href="getTDRows.html">getQuery4FB</a></td>
<td>Functions to create hypertext links that can be placed in a table cell of a HTML file</td></tr>
<tr><td style="width: 25%;"><a href="getTDRows.html">getQuery4GB</a></td>
<td>Functions to create hypertext links that can be placed in a table cell of a HTML file</td></tr>
<tr><td style="width: 25%;"><a href="annotate-defunct.html">getQuery4LL</a></td>
<td>Defunct Functions in Package 'annotate'</td></tr>
<tr><td style="width: 25%;"><a href="getTDRows.html">getQuery4OMIM</a></td>
<td>Functions to create hypertext links that can be placed in a table cell of a HTML file</td></tr>
<tr><td style="width: 25%;"><a href="getTDRows.html">getQuery4SP</a></td>
<td>Functions to create hypertext links that can be placed in a table cell of a HTML file</td></tr>
<tr><td style="width: 25%;"><a href="getTDRows.html">getQuery4TR</a></td>
<td>Functions to create hypertext links that can be placed in a table cell of a HTML file</td></tr>
<tr><td style="width: 25%;"><a href="getTDRows.html">getQuery4UG</a></td>
<td>Functions to create hypertext links that can be placed in a table cell of a HTML file</td></tr>
<tr><td style="width: 25%;"><a href="getTDRows.html">getQueryLink</a></td>
<td>Functions to create hypertext links that can be placed in a table cell of a HTML file</td></tr>
<tr><td style="width: 25%;"><a href="setRepository.html">getRepositories</a></td>
<td>Functions to add arbitrary repositories</td></tr>
<tr><td style="width: 25%;"><a href="readGEOAnn.html">getSAGEFileInfo</a></td>
<td>Function to extract data from the GEO web site</td></tr>
<tr><td style="width: 25%;"><a href="readGEOAnn.html">getSAGEGPL</a></td>
<td>Function to extract data from the GEO web site</td></tr>
<tr><td style="width: 25%;"><a href="getSeq4Acc.html">getSEQ</a></td>
<td>Queries the NCBI database to obtain the sequence for a given GenBank Accession number</td></tr>
<tr><td style="width: 25%;"><a href="getSYMBOL.html">getSYMBOL</a></td>
<td>Functions to deal with Data Packages</td></tr>
<tr><td style="width: 25%;"><a href="getTDRows.html">getTDRows</a></td>
<td>Functions to create hypertext links that can be placed in a table cell of a HTML file</td></tr>
<tr><td style="width: 25%;"><a href="getSYMBOL.html">getUniqAnnItem</a></td>
<td>Functions to deal with Data Packages</td></tr>
<tr><td style="width: 25%;"><a href="findNeighbors.html">getValidChr</a></td>
<td>A function to locate neighboring genes within a defined range around a target gene represented by a Entrez Gene ID</td></tr>
<tr><td style="width: 25%;"><a href="GO2heatmap.html">GO2heatmap</a></td>
<td>Compute a heatmap for the specified data, for either a GO category or a KEGG pathway.</td></tr>
<tr><td style="width: 25%;"><a href="GOmnplot.html">GOmnplot</a></td>
<td>A function to plot by group means against each other.</td></tr>
</table>

<h2><a name="H">-- H --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="hasGOannote.html">hasGOannote</a></td>
<td>Check for GO annotation</td></tr>
<tr><td style="width: 25%;"><a href="hgByChroms.html">hgByChroms</a></td>
<td>A dataset to show the human genome base pair locations per chromosome.</td></tr>
<tr><td style="width: 25%;"><a href="hgCLengths.html">hgCLengths</a></td>
<td>A dataset which contains the lengths (in base pairs) of the human chromosomes.</td></tr>
<tr><td style="width: 25%;"><a href="LL2homology.html">HGID2homology</a></td>
<td>DEPRECATED Functions that find the homology data for a given set of LocusLink ids or HomoloGeneIDs</td></tr>
<tr><td style="width: 25%;"><a href="hgu95Achroloc.html">hgu95Achroloc</a></td>
<td>Annotation data for the Affymetrix HGU95A GeneChip</td></tr>
<tr><td style="width: 25%;"><a href="hgu95Achrom.html">hgu95Achrom</a></td>
<td>Annotation data for the Affymetrix HGU95A GeneChip</td></tr>
<tr><td style="width: 25%;"><a href="hgu95All.html">hgu95All</a></td>
<td>Annotation data for the Affymetrix HGU95A GeneChip</td></tr>
<tr><td style="width: 25%;"><a href="hgu95AProbLocs.html">hgu95AProbLocs</a></td>
<td>chromLocation instance hgu95AProbLocs, an example of a chromLocation object</td></tr>
<tr><td style="width: 25%;"><a href="hgu95Asym.html">hgu95Asym</a></td>
<td>Annotation data for the Affymetrix HGU95A GeneChip</td></tr>
<tr><td style="width: 25%;"><a href="homoData-class.html">homoACC</a></td>
<td>Class "homoData"</td></tr>
<tr><td style="width: 25%;"><a href="homoData-class.html">homoACC-method</a></td>
<td>Class "homoData"</td></tr>
<tr><td style="width: 25%;"><a href="homoData-class.html">homoData</a></td>
<td>Class "homoData"</td></tr>
<tr><td style="width: 25%;"><a href="homoData-class.html">homoData-class</a></td>
<td>Class "homoData"</td></tr>
<tr><td style="width: 25%;"><a href="homoData-class.html">homoHGID</a></td>
<td>Class "homoData"</td></tr>
<tr><td style="width: 25%;"><a href="homoData-class.html">homoHGID-method</a></td>
<td>Class "homoData"</td></tr>
<tr><td style="width: 25%;"><a href="homoData-class.html">homoLL</a></td>
<td>Class "homoData"</td></tr>
<tr><td style="width: 25%;"><a href="homoData-class.html">homoLL-method</a></td>
<td>Class "homoData"</td></tr>
<tr><td style="width: 25%;"><a href="homoData-class.html">homoOrg</a></td>
<td>Class "homoData"</td></tr>
<tr><td style="width: 25%;"><a href="homoData-class.html">homoOrg-method</a></td>
<td>Class "homoData"</td></tr>
<tr><td style="width: 25%;"><a href="homoData-class.html">homoPS</a></td>
<td>Class "homoData"</td></tr>
<tr><td style="width: 25%;"><a href="homoData-class.html">homoPS-method</a></td>
<td>Class "homoData"</td></tr>
<tr><td style="width: 25%;"><a href="homoData-class.html">homoType</a></td>
<td>Class "homoData"</td></tr>
<tr><td style="width: 25%;"><a href="homoData-class.html">homoType-method</a></td>
<td>Class "homoData"</td></tr>
<tr><td style="width: 25%;"><a href="homoData-class.html">homoURL</a></td>
<td>Class "homoData"</td></tr>
<tr><td style="width: 25%;"><a href="homoData-class.html">homoURL-method</a></td>
<td>Class "homoData"</td></tr>
<tr><td style="width: 25%;"><a href="HTMLPage-class.html">HTMLPage</a></td>
<td>Classes to represent HTML pages</td></tr>
<tr><td style="width: 25%;"><a href="htmlpage.html">htmlpage</a></td>
<td>Functions to build HTML pages</td></tr>
<tr><td style="width: 25%;"><a href="HTMLPage-class.html">HTMLPage-class</a></td>
<td>Classes to represent HTML pages</td></tr>
<tr><td style="width: 25%;"><a href="HTMLPage-class.html">HTMLPage-method</a></td>
<td>Classes to represent HTML pages</td></tr>
</table>

<h2><a name="I">-- I --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="HTMLPage-class.html">initialize-method</a></td>
<td>Classes to represent HTML pages</td></tr>
<tr><td style="width: 25%;"><a href="isValidkey.html">isValidKey</a></td>
<td>Get or verify valid IDs for a package or OrgDb object.</td></tr>
<tr><td style="width: 25%;"><a href="isValidkey.html">isValidKey-method</a></td>
<td>Get or verify valid IDs for a package or OrgDb object.</td></tr>
</table>

<h2><a name="J">-- J --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="pubMedAbst-class.html">journal</a></td>
<td>Class pubMedAbst, a class to handle PubMed abstracts, and methods for processing them.</td></tr>
<tr><td style="width: 25%;"><a href="pubMedAbst-class.html">journal-method</a></td>
<td>Class pubMedAbst, a class to handle PubMed abstracts, and methods for processing them.</td></tr>
</table>

<h2><a name="K">-- K --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="GO2heatmap.html">KEGG2heatmap</a></td>
<td>Compute a heatmap for the specified data, for either a GO category or a KEGG pathway.</td></tr>
<tr><td style="width: 25%;"><a href="GO2heatmap.html">KEGG2heatmap-method</a></td>
<td>Compute a heatmap for the specified data, for either a GO category or a KEGG pathway.</td></tr>
<tr><td style="width: 25%;"><a href="GOmnplot.html">KEGGmnplot</a></td>
<td>A function to plot by group means against each other.</td></tr>
<tr><td style="width: 25%;"><a href="GOmnplot.html">KEGGmnplot-method</a></td>
<td>A function to plot by group means against each other.</td></tr>
</table>

<h2><a name="L">-- L --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="LL2homology.html">LL2homology</a></td>
<td>DEPRECATED Functions that find the homology data for a given set of LocusLink ids or HomoloGeneIDs</td></tr>
<tr><td style="width: 25%;"><a href="getSYMBOL.html">lookUp</a></td>
<td>Functions to deal with Data Packages</td></tr>
</table>

<h2><a name="M">-- M --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="HTMLPage-class.html">mainPage</a></td>
<td>Classes to represent HTML pages</td></tr>
<tr><td style="width: 25%;"><a href="HTMLPage-class.html">mainPage-method</a></td>
<td>Classes to represent HTML pages</td></tr>
<tr><td style="width: 25%;"><a href="makeAnchor.html">makeAnchor</a></td>
<td>A Function To Generate HTML Anchors</td></tr>
<tr><td style="width: 25%;"><a href="mapOrgs.html">mapOrgs</a></td>
<td>Functions to map to organism IDs used by NCBI homology.</td></tr>
</table>

<h2><a name="N">-- N --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="chromLocation-class.html">nChrom</a></td>
<td>Class chromLocation, a class for describing genes and their chromosome mappings.</td></tr>
<tr><td style="width: 25%;"><a href="chromLocation-class.html">nChrom-method</a></td>
<td>Class chromLocation, a class for describing genes and their chromosome mappings.</td></tr>
<tr><td style="width: 25%;"><a href="annotate-defunct.html">neighborGeneFinder</a></td>
<td>Defunct Functions in Package 'annotate'</td></tr>
</table>

<h2><a name="O">-- O --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="organism.html">organism</a></td>
<td>Convenience function for getting the organism from an object or package</td></tr>
<tr><td style="width: 25%;"><a href="chromLocation-class.html">organism-method</a></td>
<td>Class chromLocation, a class for describing genes and their chromosome mappings.</td></tr>
<tr><td style="width: 25%;"><a href="organism.html">organism-method</a></td>
<td>Convenience function for getting the organism from an object or package</td></tr>
<tr><td style="width: 25%;"><a href="homoData-class.html">orgNameNCode</a></td>
<td>Class "homoData"</td></tr>
</table>

<h2><a name="P">-- P --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="p2LL.html">p2LL</a></td>
<td>A function to map from probes to unique Entrez Gene IDs</td></tr>
<tr><td style="width: 25%;"><a href="HTMLPage-class.html">pageText</a></td>
<td>Classes to represent HTML pages</td></tr>
<tr><td style="width: 25%;"><a href="HTMLPage-class.html">pageText-method</a></td>
<td>Classes to represent HTML pages</td></tr>
<tr><td style="width: 25%;"><a href="HTMLPage-class.html">pageTitle</a></td>
<td>Classes to represent HTML pages</td></tr>
<tr><td style="width: 25%;"><a href="HTMLPage-class.html">pageTitle-method</a></td>
<td>Classes to represent HTML pages</td></tr>
<tr><td style="width: 25%;"><a href="pm.abstGrep.html">pm.abstGrep</a></td>
<td>An interface to grep for PubMed abstracts.</td></tr>
<tr><td style="width: 25%;"><a href="pm.getabst.html">pm.getabst</a></td>
<td>Obtain the abstracts for a set PubMed list.</td></tr>
<tr><td style="width: 25%;"><a href="pm.titles.html">pm.titles</a></td>
<td>Obtain the titles of the PubMed abstracts.</td></tr>
<tr><td style="width: 25%;"><a href="pmAbst2HTML.html">pmAbst2HTML</a></td>
<td>HTML Generation for PubMed Abstracts</td></tr>
<tr><td style="width: 25%;"><a href="pubMedAbst-class.html">pmid</a></td>
<td>Class pubMedAbst, a class to handle PubMed abstracts, and methods for processing them.</td></tr>
<tr><td style="width: 25%;"><a href="pubMedAbst-class.html">pmid-method</a></td>
<td>Class pubMedAbst, a class to handle PubMed abstracts, and methods for processing them.</td></tr>
<tr><td style="width: 25%;"><a href="pmid2MIAME.html">pmid2MIAME</a></td>
<td>use web to populate MIAME instance with pubmed details</td></tr>
<tr><td style="width: 25%;"><a href="PMIDAmat.html">PMIDAmat</a></td>
<td>A function to compute the probe to PubMed id incidence matrix.</td></tr>
<tr><td style="width: 25%;"><a href="pmidQuery.html">pmidQuery</a></td>
<td>A function to query PubMed</td></tr>
<tr><td style="width: 25%;"><a href="annotate-defunct.html">probesByLL</a></td>
<td>Defunct Functions in Package 'annotate'</td></tr>
<tr><td style="width: 25%;"><a href="chromLocation-class.html">probesToChrom</a></td>
<td>Class chromLocation, a class for describing genes and their chromosome mappings.</td></tr>
<tr><td style="width: 25%;"><a href="chromLocation-class.html">probesToChrom-method</a></td>
<td>Class chromLocation, a class for describing genes and their chromosome mappings.</td></tr>
<tr><td style="width: 25%;"><a href="pubMedAbst-class.html">pubDate</a></td>
<td>Class pubMedAbst, a class to handle PubMed abstracts, and methods for processing them.</td></tr>
<tr><td style="width: 25%;"><a href="pubMedAbst-class.html">pubDate-method</a></td>
<td>Class pubMedAbst, a class to handle PubMed abstracts, and methods for processing them.</td></tr>
<tr><td style="width: 25%;"><a href="pubmed.html">pubmed</a></td>
<td>A function to open the browser to Pubmed with the selected gene.</td></tr>
<tr><td style="width: 25%;"><a href="pubMedAbst-class.html">pubMedAbst</a></td>
<td>Class pubMedAbst, a class to handle PubMed abstracts, and methods for processing them.</td></tr>
<tr><td style="width: 25%;"><a href="pubMedAbst-class.html">pubMedAbst-class</a></td>
<td>Class pubMedAbst, a class to handle PubMed abstracts, and methods for processing them.</td></tr>
<tr><td style="width: 25%;"><a href="pubMedAbst-class.html">pubMedAbst-method</a></td>
<td>Class pubMedAbst, a class to handle PubMed abstracts, and methods for processing them.</td></tr>
<tr><td style="width: 25%;"><a href="PWAmat.html">PWAmat</a></td>
<td>A function to compute the probe to KEGG pathway incidence matrix.</td></tr>
</table>

<h2><a name="R">-- R --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="readGEOAnn.html">readGEOAnn</a></td>
<td>Function to extract data from the GEO web site</td></tr>
<tr><td style="width: 25%;"><a href="readGEOAnn.html">readIDNAcc</a></td>
<td>Function to extract data from the GEO web site</td></tr>
<tr><td style="width: 25%;"><a href="readGEOAnn.html">readUrl</a></td>
<td>Function to extract data from the GEO web site</td></tr>
</table>

<h2><a name="S">-- S --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="serializeEnv.html">serializeDataPkgEnvs</a></td>
<td>A Function To Serialize Environment</td></tr>
<tr><td style="width: 25%;"><a href="serializeEnv.html">serializeEnv</a></td>
<td>A Function To Serialize Environment</td></tr>
<tr><td style="width: 25%;"><a href="setRepository.html">setRepository</a></td>
<td>Functions to add arbitrary repositories</td></tr>
<tr><td style="width: 25%;"><a href="HTMLPage-class.html">show-method</a></td>
<td>Classes to represent HTML pages</td></tr>
<tr><td style="width: 25%;"><a href="chromLocation-class.html">show-method</a></td>
<td>Class chromLocation, a class for describing genes and their chromosome mappings.</td></tr>
<tr><td style="width: 25%;"><a href="homoData-class.html">show-method</a></td>
<td>Class "homoData"</td></tr>
<tr><td style="width: 25%;"><a href="pubMedAbst-class.html">show-method</a></td>
<td>Class pubMedAbst, a class to handle PubMed abstracts, and methods for processing them.</td></tr>
<tr><td style="width: 25%;"><a href="HTMLPage-class.html">sidePage</a></td>
<td>Classes to represent HTML pages</td></tr>
<tr><td style="width: 25%;"><a href="HTMLPage-class.html">sidePage-method</a></td>
<td>Classes to represent HTML pages</td></tr>
</table>

<h2><a name="T">-- T --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="HTMLPage-class.html">toFile</a></td>
<td>Classes to represent HTML pages</td></tr>
<tr><td style="width: 25%;"><a href="HTMLPage-class.html">toFile-method</a></td>
<td>Classes to represent HTML pages</td></tr>
<tr><td style="width: 25%;"><a href="HTMLPage-class.html">topPage</a></td>
<td>Classes to represent HTML pages</td></tr>
<tr><td style="width: 25%;"><a href="HTMLPage-class.html">topPage-method</a></td>
<td>Classes to represent HTML pages</td></tr>
</table>

<h2><a name="U">-- U --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="UniGeneQuery.html">UniGeneQuery</a></td>
<td>Create a Query String for a UniGene Identifier</td></tr>
<tr><td style="width: 25%;"><a href="updateSymbolsToValidKeys.html">updateSymbolsToValidKeys</a></td>
<td>Take a list of symbols and translate them into the best possible ID for a package.</td></tr>
<tr><td style="width: 25%;"><a href="usedChromGenes.html">usedChromGenes</a></td>
<td>A function to select used genes on a chromosome from an ExpressionSet.</td></tr>
</table>

<h2><a name="W">-- W --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="findNeighbors.html">weightByConfi</a></td>
<td>A function to locate neighboring genes within a defined range around a target gene represented by a Entrez Gene ID</td></tr>
<tr><td style="width: 25%;"><a href="ACCNUMStats.html">whatACC</a></td>
<td>Provides statistics on the types of ids used for the ACCNUM environment of a given data package</td></tr>
</table>
</body></html>
