<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head><title>R: Analyses of Phylogenetics and Evolution</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="stylesheet" type="text/css" href="R.css" />
</head><body>
<h1> Analyses of Phylogenetics and Evolution
<img class="toplogo" src="../../../doc/html/Rlogo.svg" alt="[R logo]" />
</h1>
<hr/>
<div style="text-align: center;">
<a href="../../../doc/html/packages.html"><img class="arrow" src="../../../doc/html/left.jpg" alt="[Up]" /></a>
<a href="../../../doc/html/index.html"><img class="arrow" src="../../../doc/html/up.jpg" alt="[Top]" /></a>
</div><h2>Documentation for package &lsquo;ape&rsquo; version 5.2</h2>

<ul><li><a href="../DESCRIPTION">DESCRIPTION file</a>.</li>
<li><a href="../doc/index.html">User guides, package vignettes and other documentation.</a></li>
<li><a href="../NEWS">Package NEWS</a>.</li>
</ul>

<h2>Help Pages</h2>


<p style="text-align: center;">
<a href="# "> </a>
<a href="#A">A</a>
<a href="#B">B</a>
<a href="#C">C</a>
<a href="#D">D</a>
<a href="#E">E</a>
<a href="#F">F</a>
<a href="#G">G</a>
<a href="#H">H</a>
<a href="#I">I</a>
<a href="#K">K</a>
<a href="#L">L</a>
<a href="#M">M</a>
<a href="#N">N</a>
<a href="#O">O</a>
<a href="#P">P</a>
<a href="#R">R</a>
<a href="#S">S</a>
<a href="#T">T</a>
<a href="#U">U</a>
<a href="#V">V</a>
<a href="#W">W</a>
<a href="#X">X</a>
<a href="#Y">Y</a>
<a href="#Z">Z</a>
<a href="#misc">misc</a>
</p>

<table width="100%">
<tr><td style="width: 25%;"><a href="ape-package.html">ape-package</a></td>
<td>Analyses of Phylogenetics and Evolution</td></tr>
</table>

<h2><a name="A">-- A --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="AAbin.html">AAbin</a></td>
<td>Amino Acid Sequences</td></tr>
<tr><td style="width: 25%;"><a href="AAbin.html">AAsubst</a></td>
<td>Amino Acid Sequences</td></tr>
<tr><td style="width: 25%;"><a href="label2table.html">abbreviateGenus</a></td>
<td>Label Management</td></tr>
<tr><td style="width: 25%;"><a href="ace.html">ace</a></td>
<td>Ancestral Character Estimation</td></tr>
<tr><td style="width: 25%;"><a href="add.scale.bar.html">add.scale.bar</a></td>
<td>Add a Scale Bar to a Phylogeny Plot</td></tr>
<tr><td style="width: 25%;"><a href="additive.html">additive</a></td>
<td>Incomplete Distance Matrix Filling</td></tr>
<tr><td style="width: 25%;"><a href="ace.html">AIC.ace</a></td>
<td>Ancestral Character Estimation</td></tr>
<tr><td style="width: 25%;"><a href="alex.html">alex</a></td>
<td>Alignment Explorer With Multiple Devices</td></tr>
<tr><td style="width: 25%;"><a href="all.equal.DNAbin.html">all.equal.DNAbin</a></td>
<td>Compare DNA Sets</td></tr>
<tr><td style="width: 25%;"><a href="all.equal.phylo.html">all.equal.phylo</a></td>
<td>Global Comparison of two Phylogenies</td></tr>
<tr><td style="width: 25%;"><a href="alview.html">alview</a></td>
<td>Print DNA or AA Sequence Alignement</td></tr>
<tr><td style="width: 25%;"><a href="ace.html">anova.ace</a></td>
<td>Ancestral Character Estimation</td></tr>
<tr><td style="width: 25%;"><a href="ape-package.html">ape</a></td>
<td>Analyses of Phylogenetics and Evolution</td></tr>
<tr><td style="width: 25%;"><a href="apetools.html">apetools</a></td>
<td>Tools to Explore Files</td></tr>
<tr><td style="width: 25%;"><a href="is.compatible.html">arecompatible</a></td>
<td>Check Compatibility of Splits</td></tr>
<tr><td style="width: 25%;"><a href="AAbin.html">as.AAbin</a></td>
<td>Amino Acid Sequences</td></tr>
<tr><td style="width: 25%;"><a href="AAbin.html">as.AAbin.AAMultipleAlignment</a></td>
<td>Amino Acid Sequences</td></tr>
<tr><td style="width: 25%;"><a href="AAbin.html">as.AAbin.AAString</a></td>
<td>Amino Acid Sequences</td></tr>
<tr><td style="width: 25%;"><a href="AAbin.html">as.AAbin.AAStringSet</a></td>
<td>Amino Acid Sequences</td></tr>
<tr><td style="width: 25%;"><a href="AAbin.html">as.AAbin.character</a></td>
<td>Amino Acid Sequences</td></tr>
<tr><td style="width: 25%;"><a href="AAbin.html">as.AAbin.list</a></td>
<td>Amino Acid Sequences</td></tr>
<tr><td style="width: 25%;"><a href="as.alignment.html">as.alignment</a></td>
<td>Conversion Among DNA Sequence Internal Formats</td></tr>
<tr><td style="width: 25%;"><a href="as.bitsplits.html">as.bitsplits</a></td>
<td>Split Frequencies and Conversion Among Split Classes</td></tr>
<tr><td style="width: 25%;"><a href="as.bitsplits.html">as.bitsplits.prop.part</a></td>
<td>Split Frequencies and Conversion Among Split Classes</td></tr>
<tr><td style="width: 25%;"><a href="AAbin.html">as.character.AAbin</a></td>
<td>Amino Acid Sequences</td></tr>
<tr><td style="width: 25%;"><a href="as.alignment.html">as.character.DNAbin</a></td>
<td>Conversion Among DNA Sequence Internal Formats</td></tr>
<tr><td style="width: 25%;"><a href="as.alignment.html">as.DNAbin</a></td>
<td>Conversion Among DNA Sequence Internal Formats</td></tr>
<tr><td style="width: 25%;"><a href="as.alignment.html">as.DNAbin.alignment</a></td>
<td>Conversion Among DNA Sequence Internal Formats</td></tr>
<tr><td style="width: 25%;"><a href="as.alignment.html">as.DNAbin.character</a></td>
<td>Conversion Among DNA Sequence Internal Formats</td></tr>
<tr><td style="width: 25%;"><a href="as.alignment.html">as.DNAbin.DNAMultipleAlignment</a></td>
<td>Conversion Among DNA Sequence Internal Formats</td></tr>
<tr><td style="width: 25%;"><a href="as.alignment.html">as.DNAbin.DNAString</a></td>
<td>Conversion Among DNA Sequence Internal Formats</td></tr>
<tr><td style="width: 25%;"><a href="as.alignment.html">as.DNAbin.DNAStringSet</a></td>
<td>Conversion Among DNA Sequence Internal Formats</td></tr>
<tr><td style="width: 25%;"><a href="as.alignment.html">as.DNAbin.list</a></td>
<td>Conversion Among DNA Sequence Internal Formats</td></tr>
<tr><td style="width: 25%;"><a href="as.alignment.html">as.DNAbin.PairwiseAlignmentsSingleSubject</a></td>
<td>Conversion Among DNA Sequence Internal Formats</td></tr>
<tr><td style="width: 25%;"><a href="evonet.html">as.evonet</a></td>
<td>Evolutionary Networks</td></tr>
<tr><td style="width: 25%;"><a href="evonet.html">as.evonet.phylo</a></td>
<td>Evolutionary Networks</td></tr>
<tr><td style="width: 25%;"><a href="as.phylo.html">as.hclust.phylo</a></td>
<td>Conversion Among Tree and Network Objects</td></tr>
<tr><td style="width: 25%;"><a href="as.phylo.html">as.igraph</a></td>
<td>Conversion Among Tree and Network Objects</td></tr>
<tr><td style="width: 25%;"><a href="evonet.html">as.igraph.evonet</a></td>
<td>Evolutionary Networks</td></tr>
<tr><td style="width: 25%;"><a href="as.phylo.html">as.igraph.phylo</a></td>
<td>Conversion Among Tree and Network Objects</td></tr>
<tr><td style="width: 25%;"><a href="AAbin.html">as.list.AAbin</a></td>
<td>Amino Acid Sequences</td></tr>
<tr><td style="width: 25%;"><a href="DNAbin.html">as.list.DNAbin</a></td>
<td>Manipulate DNA Sequences in Bit-Level Format</td></tr>
<tr><td style="width: 25%;"><a href="as.matching.html">as.matching</a></td>
<td>Conversion Between Phylo and Matching Objects</td></tr>
<tr><td style="width: 25%;"><a href="as.matching.html">as.matching.phylo</a></td>
<td>Conversion Between Phylo and Matching Objects</td></tr>
<tr><td style="width: 25%;"><a href="AAbin.html">as.matrix.AAbin</a></td>
<td>Amino Acid Sequences</td></tr>
<tr><td style="width: 25%;"><a href="DNAbin.html">as.matrix.DNAbin</a></td>
<td>Manipulate DNA Sequences in Bit-Level Format</td></tr>
<tr><td style="width: 25%;"><a href="evonet.html">as.network.evonet</a></td>
<td>Evolutionary Networks</td></tr>
<tr><td style="width: 25%;"><a href="as.phylo.html">as.network.phylo</a></td>
<td>Conversion Among Tree and Network Objects</td></tr>
<tr><td style="width: 25%;"><a href="evonet.html">as.networx.evonet</a></td>
<td>Evolutionary Networks</td></tr>
<tr><td style="width: 25%;"><a href="AAbin.html">as.phyDat.AAbin</a></td>
<td>Amino Acid Sequences</td></tr>
<tr><td style="width: 25%;"><a href="as.phylo.html">as.phylo</a></td>
<td>Conversion Among Tree and Network Objects</td></tr>
<tr><td style="width: 25%;"><a href="evonet.html">as.phylo.evonet</a></td>
<td>Evolutionary Networks</td></tr>
<tr><td style="width: 25%;"><a href="as.phylo.formula.html">as.phylo.formula</a></td>
<td>Conversion from Taxonomy Variables to Phylogenetic Trees</td></tr>
<tr><td style="width: 25%;"><a href="as.phylo.html">as.phylo.hclust</a></td>
<td>Conversion Among Tree and Network Objects</td></tr>
<tr><td style="width: 25%;"><a href="as.matching.html">as.phylo.matching</a></td>
<td>Conversion Between Phylo and Matching Objects</td></tr>
<tr><td style="width: 25%;"><a href="as.phylo.html">as.phylo.phylog</a></td>
<td>Conversion Among Tree and Network Objects</td></tr>
<tr><td style="width: 25%;"><a href="as.bitsplits.html">as.prop.part</a></td>
<td>Split Frequencies and Conversion Among Split Classes</td></tr>
<tr><td style="width: 25%;"><a href="as.bitsplits.html">as.prop.part.bitsplits</a></td>
<td>Split Frequencies and Conversion Among Split Classes</td></tr>
<tr><td style="width: 25%;"><a href="axisPhylo.html">axisPhylo</a></td>
<td>Axis on Side of Phylogeny</td></tr>
</table>

<h2><a name="B">-- B --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="balance.html">balance</a></td>
<td>Balance of a Dichotomous Phylogenetic Tree</td></tr>
<tr><td style="width: 25%;"><a href="base.freq.html">base.freq</a></td>
<td>Base frequencies from DNA Sequences</td></tr>
<tr><td style="width: 25%;"><a href="bd.ext.html">bd.ext</a></td>
<td>Extended Version of the Birth-Death Models to Estimate Speciation and Extinction Rates</td></tr>
<tr><td style="width: 25%;"><a href="bd.time.html">bd.time</a></td>
<td>Time-Dependent Birth-Death Models</td></tr>
<tr><td style="width: 25%;"><a href="binaryPGLMM.html">binaryPGLMM</a></td>
<td>Phylogenetic Generalized Linear Mixed Model for Binary Data</td></tr>
<tr><td style="width: 25%;"><a href="binaryPGLMM.html">binaryPGLMM.sim</a></td>
<td>Phylogenetic Generalized Linear Mixed Model for Binary Data</td></tr>
<tr><td style="width: 25%;"><a href="bind.tree.html">bind.tree</a></td>
<td>Binds Trees</td></tr>
<tr><td style="width: 25%;"><a href="bionj.html">bionj</a></td>
<td>Tree Estimation Based on an Improved Version of the NJ Algorithm</td></tr>
<tr><td style="width: 25%;"><a href="njs.html">bionjs</a></td>
<td>Tree Reconstruction from Incomplete Distances With NJ* or bio-NJ*</td></tr>
<tr><td style="width: 25%;"><a href="pcoa.html">biplot.pcoa</a></td>
<td>Principal Coordinate Analysis</td></tr>
<tr><td style="width: 25%;"><a href="bird.families.html">bird.families</a></td>
<td>Phylogeny of the Families of Birds From Sibley and Ahlquist</td></tr>
<tr><td style="width: 25%;"><a href="bird.orders.html">bird.orders</a></td>
<td>Phylogeny of the Orders of Birds From Sibley and Ahlquist</td></tr>
<tr><td style="width: 25%;"><a href="birthdeath.html">birthdeath</a></td>
<td>Estimation of Speciation and Extinction Rates With Birth-Death Models</td></tr>
<tr><td style="width: 25%;"><a href="as.bitsplits.html">bitsplits</a></td>
<td>Split Frequencies and Conversion Among Split Classes</td></tr>
<tr><td style="width: 25%;"><a href="boot.phylo.html">boot.phylo</a></td>
<td>Tree Bipartition and Bootstrapping Phylogenies</td></tr>
<tr><td style="width: 25%;"><a href="branching.times.html">branching.times</a></td>
<td>Branching Times of a Phylogenetic Tree</td></tr>
<tr><td style="width: 25%;"><a href="apetools.html">bydir</a></td>
<td>Tools to Explore Files</td></tr>
</table>

<h2><a name="C">-- C --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="AAbin.html">c.AAbin</a></td>
<td>Amino Acid Sequences</td></tr>
<tr><td style="width: 25%;"><a href="DNAbin.html">c.DNAbin</a></td>
<td>Manipulate DNA Sequences in Bit-Level Format</td></tr>
<tr><td style="width: 25%;"><a href="c.phylo.html">c.multiPhylo</a></td>
<td>Building Lists of Trees</td></tr>
<tr><td style="width: 25%;"><a href="c.phylo.html">c.phylo</a></td>
<td>Building Lists of Trees</td></tr>
<tr><td style="width: 25%;"><a href="CADM.global.html">CADM</a></td>
<td>Congruence among distance matrices</td></tr>
<tr><td style="width: 25%;"><a href="CADM.global.html">CADM.global</a></td>
<td>Congruence among distance matrices</td></tr>
<tr><td style="width: 25%;"><a href="CADM.global.html">CADM.post</a></td>
<td>Congruence among distance matrices</td></tr>
<tr><td style="width: 25%;"><a href="carnivora.html">carnivora</a></td>
<td>Carnivora body sizes and life history traits</td></tr>
<tr><td style="width: 25%;"><a href="DNAbin.html">cbind.DNAbin</a></td>
<td>Manipulate DNA Sequences in Bit-Level Format</td></tr>
<tr><td style="width: 25%;"><a href="checkAlignment.html">checkAlignment</a></td>
<td>Check DNA Alignments</td></tr>
<tr><td style="width: 25%;"><a href="checkValidPhylo.html">checkValidPhylo</a></td>
<td>Check the Structure of a "phylo" Object</td></tr>
<tr><td style="width: 25%;"><a href="cherry.html">cherry</a></td>
<td>Number of Cherries and Null Models of Trees</td></tr>
<tr><td style="width: 25%;"><a href="chiroptera.html">chiroptera</a></td>
<td>Bat Phylogeny</td></tr>
<tr><td style="width: 25%;"><a href="chronoMPL.html">chronoMPL</a></td>
<td>Molecular Dating With Mean Path Lengths</td></tr>
<tr><td style="width: 25%;"><a href="chronopl.html">chronopl</a></td>
<td>Molecular Dating With Penalized Likelihood</td></tr>
<tr><td style="width: 25%;"><a href="chronos.html">chronos</a></td>
<td>Molecular Dating by Penalised Likelihood and Maximum Likelihood</td></tr>
<tr><td style="width: 25%;"><a href="chronos.html">chronos.control</a></td>
<td>Molecular Dating by Penalised Likelihood and Maximum Likelihood</td></tr>
<tr><td style="width: 25%;"><a href="reorder.phylo.html">cladewise</a></td>
<td>Internal Reordering of Trees</td></tr>
<tr><td style="width: 25%;"><a href="clustal.html">clustal</a></td>
<td>Multiple Sequence Alignment with External Applications</td></tr>
<tr><td style="width: 25%;"><a href="clustal.html">clustalomega</a></td>
<td>Multiple Sequence Alignment with External Applications</td></tr>
<tr><td style="width: 25%;"><a href="coalescent.intervals.html">coalescent.intervals</a></td>
<td>Coalescent Intervals</td></tr>
<tr><td style="width: 25%;"><a href="coalescent.intervals.html">coalescent.intervals.default</a></td>
<td>Coalescent Intervals</td></tr>
<tr><td style="width: 25%;"><a href="coalescent.intervals.html">coalescent.intervals.phylo</a></td>
<td>Coalescent Intervals</td></tr>
<tr><td style="width: 25%;"><a href="corBlomberg.html">coef.corBlomberg</a></td>
<td>Blomberg et al.'s Correlation Structure</td></tr>
<tr><td style="width: 25%;"><a href="corBrownian.html">coef.corBrownian</a></td>
<td>Brownian Correlation Structure</td></tr>
<tr><td style="width: 25%;"><a href="corGrafen.html">coef.corGrafen</a></td>
<td>Grafen's (1989) Correlation Structure</td></tr>
<tr><td style="width: 25%;"><a href="corMartins.html">coef.corMartins</a></td>
<td>Martins's (1997) Correlation Structure</td></tr>
<tr><td style="width: 25%;"><a href="corPagel.html">coef.corPagel</a></td>
<td>Pagel's "lambda" Correlation Structure</td></tr>
<tr><td style="width: 25%;"><a href="collapse.singles.html">collapse.singles</a></td>
<td>Collapse Single Nodes</td></tr>
<tr><td style="width: 25%;"><a href="collapsed.intervals.html">collapsed.intervals</a></td>
<td>Collapsed Coalescent Intervals</td></tr>
<tr><td style="width: 25%;"><a href="compar.cheverud.html">compar.cheverud</a></td>
<td>Cheverud's Comparative Method</td></tr>
<tr><td style="width: 25%;"><a href="compar.gee.html">compar.gee</a></td>
<td>Comparative Analysis with GEEs</td></tr>
<tr><td style="width: 25%;"><a href="compar.lynch.html">compar.lynch</a></td>
<td>Lynch's Comparative Method</td></tr>
<tr><td style="width: 25%;"><a href="compar.ou.html">compar.ou</a></td>
<td>Ornstein-Uhlenbeck Model for Continuous Characters</td></tr>
<tr><td style="width: 25%;"><a href="comparePhylo.html">comparePhylo</a></td>
<td>Compare Two "phylo" Objects</td></tr>
<tr><td style="width: 25%;"><a href="trans.html">complement</a></td>
<td>Translation from DNA to Amino Acid Sequences</td></tr>
<tr><td style="width: 25%;"><a href="compute.brlen.html">compute.brlen</a></td>
<td>Branch Lengths Computation</td></tr>
<tr><td style="width: 25%;"><a href="compute.brtime.html">compute.brtime</a></td>
<td>Compute and Set Branching Times</td></tr>
<tr><td style="width: 25%;"><a href="consensus.html">consensus</a></td>
<td>Concensus Trees</td></tr>
<tr><td style="width: 25%;"><a href="cophenetic.phylo.html">cophenetic.phylo</a></td>
<td>Pairwise Distances from a Phylogenetic Tree</td></tr>
<tr><td style="width: 25%;"><a href="cophyloplot.html">cophyloplot</a></td>
<td>Plots two phylogenetic trees face to face with links between the tips.</td></tr>
<tr><td style="width: 25%;"><a href="corBlomberg.html">corBlomberg</a></td>
<td>Blomberg et al.'s Correlation Structure</td></tr>
<tr><td style="width: 25%;"><a href="corBrownian.html">corBrownian</a></td>
<td>Brownian Correlation Structure</td></tr>
<tr><td style="width: 25%;"><a href="corClasses.html">corClasses</a></td>
<td>Phylogenetic Correlation Structures</td></tr>
<tr><td style="width: 25%;"><a href="corGrafen.html">corGrafen</a></td>
<td>Grafen's (1989) Correlation Structure</td></tr>
<tr><td style="width: 25%;"><a href="corMartins.html">corMartins</a></td>
<td>Martins's (1997) Correlation Structure</td></tr>
<tr><td style="width: 25%;"><a href="corBlomberg.html">corMatrix.corBlomberg</a></td>
<td>Blomberg et al.'s Correlation Structure</td></tr>
<tr><td style="width: 25%;"><a href="corBrownian.html">corMatrix.corBrownian</a></td>
<td>Brownian Correlation Structure</td></tr>
<tr><td style="width: 25%;"><a href="corGrafen.html">corMatrix.corGrafen</a></td>
<td>Grafen's (1989) Correlation Structure</td></tr>
<tr><td style="width: 25%;"><a href="corMartins.html">corMatrix.corMartins</a></td>
<td>Martins's (1997) Correlation Structure</td></tr>
<tr><td style="width: 25%;"><a href="corPagel.html">corMatrix.corPagel</a></td>
<td>Pagel's "lambda" Correlation Structure</td></tr>
<tr><td style="width: 25%;"><a href="corPagel.html">corPagel</a></td>
<td>Pagel's "lambda" Correlation Structure</td></tr>
<tr><td style="width: 25%;"><a href="corClasses.html">corPhyl</a></td>
<td>Phylogenetic Correlation Structures</td></tr>
<tr><td style="width: 25%;"><a href="corphylo.html">corphylo</a></td>
<td>Correlations among Multiple Traits with Phylogenetic Signal</td></tr>
<tr><td style="width: 25%;"><a href="correlogram.formula.html">correlogram.formula</a></td>
<td>Phylogenetic Correlogram</td></tr>
<tr><td style="width: 25%;"><a href="as.bitsplits.html">countBipartitions</a></td>
<td>Split Frequencies and Conversion Among Split Classes</td></tr>
<tr><td style="width: 25%;"><a href="cynipids.html">cynipids</a></td>
<td>NEXUS Data Example</td></tr>
</table>

<h2><a name="D">-- D --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="cynipids.html">data.nex</a></td>
<td>NEXUS Data Example</td></tr>
<tr><td style="width: 25%;"><a href="dbd.html">dbd</a></td>
<td>Probability Density Under Birth-Death Models</td></tr>
<tr><td style="width: 25%;"><a href="dbd.html">dbdTime</a></td>
<td>Probability Density Under Birth-Death Models</td></tr>
<tr><td style="width: 25%;"><a href="def.html">def</a></td>
<td>Definition of Vectors for Plotting or Annotating</td></tr>
<tr><td style="width: 25%;"><a href="del.gaps.html">del.colgapsonly</a></td>
<td>Delete Alignment Gaps in DNA Sequences</td></tr>
<tr><td style="width: 25%;"><a href="del.gaps.html">del.gaps</a></td>
<td>Delete Alignment Gaps in DNA Sequences</td></tr>
<tr><td style="width: 25%;"><a href="del.gaps.html">del.rowgapsonly</a></td>
<td>Delete Alignment Gaps in DNA Sequences</td></tr>
<tr><td style="width: 25%;"><a href="delta.plot.html">delta.plot</a></td>
<td>Delta Plots</td></tr>
<tr><td style="width: 25%;"><a href="ace.html">deviance.ace</a></td>
<td>Ancestral Character Estimation</td></tr>
<tr><td style="width: 25%;"><a href="multi2di.html">di2multi</a></td>
<td>Collapse and Resolve Multichotomies</td></tr>
<tr><td style="width: 25%;"><a href="multi2di.html">di2multi.multiPhylo</a></td>
<td>Collapse and Resolve Multichotomies</td></tr>
<tr><td style="width: 25%;"><a href="multi2di.html">di2multi.phylo</a></td>
<td>Collapse and Resolve Multichotomies</td></tr>
<tr><td style="width: 25%;"><a href="AAbin.html">dist.aa</a></td>
<td>Amino Acid Sequences</td></tr>
<tr><td style="width: 25%;"><a href="dist.dna.html">dist.dna</a></td>
<td>Pairwise Distances from DNA Sequences</td></tr>
<tr><td style="width: 25%;"><a href="dist.gene.html">dist.gene</a></td>
<td>Pairwise Distances from Genetic Data</td></tr>
<tr><td style="width: 25%;"><a href="cophenetic.phylo.html">dist.nodes</a></td>
<td>Pairwise Distances from a Phylogenetic Tree</td></tr>
<tr><td style="width: 25%;"><a href="dist.topo.html">dist.topo</a></td>
<td>Topological Distances Between Two Trees</td></tr>
<tr><td style="width: 25%;"><a href="diversi.gof.html">diversi.gof</a></td>
<td>Tests of Constant Diversification Rates</td></tr>
<tr><td style="width: 25%;"><a href="diversi.time.html">diversi.time</a></td>
<td>Analysis of Diversification with Survival Models</td></tr>
<tr><td style="width: 25%;"><a href="diversity.contrast.test.html">diversity.contrast.test</a></td>
<td>Diversity Contrast Test</td></tr>
<tr><td style="width: 25%;"><a href="DNAbin.html">DNAbin</a></td>
<td>Manipulate DNA Sequences in Bit-Level Format</td></tr>
<tr><td style="width: 25%;"><a href="DNAbin2indel.html">DNAbin2indel</a></td>
<td>Recode Blocks of Indels</td></tr>
<tr><td style="width: 25%;"><a href="plot.phyloExtra.html">drawSupportOnEdges</a></td>
<td>Extra Fuctions to Plot and Annotate Phylogenies</td></tr>
<tr><td style="width: 25%;"><a href="rlineage.html">drop.fossil</a></td>
<td>Tree Simulation Under the Time-Dependent Birth-Death Models</td></tr>
<tr><td style="width: 25%;"><a href="drop.tip.html">drop.tip</a></td>
<td>Remove Tips in a Phylogenetic Tree</td></tr>
<tr><td style="width: 25%;"><a href="compar.gee.html">drop1.compar.gee</a></td>
<td>Comparative Analysis with GEEs</td></tr>
<tr><td style="width: 25%;"><a href="dbd.html">dyule</a></td>
<td>Probability Density Under Birth-Death Models</td></tr>
</table>

<h2><a name="E">-- E --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="nodelabels.html">edgelabels</a></td>
<td>Labelling the Nodes, Tips, and Edges of a Tree</td></tr>
<tr><td style="width: 25%;"><a href="edges.html">edges</a></td>
<td>Draw Additional Edges on a Plotted Tree</td></tr>
<tr><td style="width: 25%;"><a href="apetools.html">editFileExtensions</a></td>
<td>Tools to Explore Files</td></tr>
<tr><td style="width: 25%;"><a href="node.dating.html">estimate.dates</a></td>
<td>node.dating</td></tr>
<tr><td style="width: 25%;"><a href="node.dating.html">estimate.mu</a></td>
<td>node.dating</td></tr>
<tr><td style="width: 25%;"><a href="evonet.html">evonet</a></td>
<td>Evolutionary Networks</td></tr>
<tr><td style="width: 25%;"><a href="ewLasso.html">ewLasso</a></td>
<td>Incomplete distances and edge weights of unrooted topology</td></tr>
<tr><td style="width: 25%;"><a href="drop.tip.html">extract.clade</a></td>
<td>Remove Tips in a Phylogenetic Tree</td></tr>
<tr><td style="width: 25%;"><a href="mcmc.popsize.html">extract.popsize</a></td>
<td>Reversible Jump MCMC to Infer Demographic History</td></tr>
</table>

<h2><a name="F">-- F --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="edges.html">fancyarrows</a></td>
<td>Draw Additional Edges on a Plotted Tree</td></tr>
<tr><td style="width: 25%;"><a href="fastme.html">FastME</a></td>
<td>Tree Estimation Based on the Minimum Evolution Algorithm</td></tr>
<tr><td style="width: 25%;"><a href="fastme.html">fastme</a></td>
<td>Tree Estimation Based on the Minimum Evolution Algorithm</td></tr>
<tr><td style="width: 25%;"><a href="fastme.html">fastme.bal</a></td>
<td>Tree Estimation Based on the Minimum Evolution Algorithm</td></tr>
<tr><td style="width: 25%;"><a href="fastme.html">fastme.ols</a></td>
<td>Tree Estimation Based on the Minimum Evolution Algorithm</td></tr>
<tr><td style="width: 25%;"><a href="skyline.html">find.skyline.epsilon</a></td>
<td>Skyline Plot Estimate of Effective Population Size</td></tr>
<tr><td style="width: 25%;"><a href="base.freq.html">Ftab</a></td>
<td>Base frequencies from DNA Sequences</td></tr>
</table>

<h2><a name="G">-- G --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="gammaStat.html">gammaStat</a></td>
<td>Gamma-Statistic of Pybus and Harvey</td></tr>
<tr><td style="width: 25%;"><a href="base.freq.html">GC.content</a></td>
<td>Base frequencies from DNA Sequences</td></tr>
<tr><td style="width: 25%;"><a href="mrca.html">getMRCA</a></td>
<td>Find Most Recent Common Ancestors Between Pairs</td></tr>
<tr><td style="width: 25%;"><a href="parafit.html">gopher.D</a></td>
<td>Test of host-parasite coevolution</td></tr>
</table>

<h2><a name="H">-- H --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="collapse.singles.html">has.singles</a></td>
<td>Collapse Single Nodes</td></tr>
<tr><td style="width: 25%;"><a href="hivtree.html">hivtree</a></td>
<td>Phylogenetic Tree of 193 HIV-1 Sequences</td></tr>
<tr><td style="width: 25%;"><a href="hivtree.html">hivtree.newick</a></td>
<td>Phylogenetic Tree of 193 HIV-1 Sequences</td></tr>
<tr><td style="width: 25%;"><a href="hivtree.html">hivtree.table</a></td>
<td>Phylogenetic Tree of 193 HIV-1 Sequences</td></tr>
<tr><td style="width: 25%;"><a href="howmanytrees.html">howmanytrees</a></td>
<td>Calculate Numbers of Phylogenetic Trees</td></tr>
<tr><td style="width: 25%;"><a href="parafit.html">HP.links</a></td>
<td>Test of host-parasite coevolution</td></tr>
</table>

<h2><a name="I">-- I --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="identify.phylo.html">identify.phylo</a></td>
<td>Graphical Identification of Nodes and Tips</td></tr>
<tr><td style="width: 25%;"><a href="AAbin.html">image.AAbin</a></td>
<td>Amino Acid Sequences</td></tr>
<tr><td style="width: 25%;"><a href="image.DNAbin.html">image.DNAbin</a></td>
<td>Plot of DNA Sequence Alignement</td></tr>
<tr><td style="width: 25%;"><a href="Initialize.corPhyl.html">Initialize.corPhyl</a></td>
<td>Initialize a 'corPhyl' Structure Object</td></tr>
<tr><td style="width: 25%;"><a href="is.binary.tree.html">is.binary</a></td>
<td>Test for Binary Tree</td></tr>
<tr><td style="width: 25%;"><a href="is.binary.tree.html">is.binary.multiPhylo</a></td>
<td>Test for Binary Tree</td></tr>
<tr><td style="width: 25%;"><a href="is.binary.tree.html">is.binary.phylo</a></td>
<td>Test for Binary Tree</td></tr>
<tr><td style="width: 25%;"><a href="is.binary.tree.html">is.binary.tree</a></td>
<td>Test for Binary Tree</td></tr>
<tr><td style="width: 25%;"><a href="is.compatible.html">is.compatible</a></td>
<td>Check Compatibility of Splits</td></tr>
<tr><td style="width: 25%;"><a href="is.compatible.html">is.compatible.bitsplits</a></td>
<td>Check Compatibility of Splits</td></tr>
<tr><td style="width: 25%;"><a href="is.monophyletic.html">is.monophyletic</a></td>
<td>Is Group Monophyletic</td></tr>
<tr><td style="width: 25%;"><a href="root.html">is.rooted</a></td>
<td>Roots Phylogenetic Trees</td></tr>
<tr><td style="width: 25%;"><a href="root.html">is.rooted.multiPhylo</a></td>
<td>Roots Phylogenetic Trees</td></tr>
<tr><td style="width: 25%;"><a href="root.html">is.rooted.phylo</a></td>
<td>Roots Phylogenetic Trees</td></tr>
<tr><td style="width: 25%;"><a href="is.ultrametric.html">is.ultrametric</a></td>
<td>Test if a Tree is Ultrametric</td></tr>
<tr><td style="width: 25%;"><a href="is.ultrametric.html">is.ultrametric.multiPhylo</a></td>
<td>Test if a Tree is Ultrametric</td></tr>
<tr><td style="width: 25%;"><a href="is.ultrametric.html">is.ultrametric.phylo</a></td>
<td>Test if a Tree is Ultrametric</td></tr>
</table>

<h2><a name="K">-- K --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="drop.tip.html">keep.tip</a></td>
<td>Remove Tips in a Phylogenetic Tree</td></tr>
<tr><td style="width: 25%;"><a href="kronoviz.html">kronoviz</a></td>
<td>Plot Multiple Chronograms on the Same Scale</td></tr>
</table>

<h2><a name="L">-- L --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="label2table.html">label2table</a></td>
<td>Label Management</td></tr>
<tr><td style="width: 25%;"><a href="AAbin.html">labels.AAbin</a></td>
<td>Amino Acid Sequences</td></tr>
<tr><td style="width: 25%;"><a href="DNAbin.html">labels.DNAbin</a></td>
<td>Manipulate DNA Sequences in Bit-Level Format</td></tr>
<tr><td style="width: 25%;"><a href="ladderize.html">ladderize</a></td>
<td>Ladderize a Tree</td></tr>
<tr><td style="width: 25%;"><a href="parafit.html">lice.D</a></td>
<td>Test of host-parasite coevolution</td></tr>
<tr><td style="width: 25%;"><a href="mcmc.popsize.html">lines.popsize</a></td>
<td>Reversible Jump MCMC to Infer Demographic History</td></tr>
<tr><td style="width: 25%;"><a href="skylineplot.html">lines.skyline</a></td>
<td>Drawing Skyline Plot Graphs</td></tr>
<tr><td style="width: 25%;"><a href="lmorigin.html">lmorigin</a></td>
<td>Multiple regression through the origin</td></tr>
<tr><td style="width: 25%;"><a href="lmorigin.html">lmorigin.ex1</a></td>
<td>Multiple regression through the origin</td></tr>
<tr><td style="width: 25%;"><a href="lmorigin.html">lmorigin.ex2</a></td>
<td>Multiple regression through the origin</td></tr>
<tr><td style="width: 25%;"><a href="ace.html">logLik.ace</a></td>
<td>Ancestral Character Estimation</td></tr>
<tr><td style="width: 25%;"><a href="LTT.html">LTT</a></td>
<td>Theoretical Lineage-Through Time Plots</td></tr>
<tr><td style="width: 25%;"><a href="ltt.plot.html">ltt.coplot</a></td>
<td>Lineages Through Time Plot</td></tr>
<tr><td style="width: 25%;"><a href="ltt.plot.html">ltt.lines</a></td>
<td>Lineages Through Time Plot</td></tr>
<tr><td style="width: 25%;"><a href="ltt.plot.html">ltt.plot</a></td>
<td>Lineages Through Time Plot</td></tr>
<tr><td style="width: 25%;"><a href="ltt.plot.html">ltt.plot.coords</a></td>
<td>Lineages Through Time Plot</td></tr>
</table>

<h2><a name="M">-- M --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="chronos.html">makeChronosCalib</a></td>
<td>Molecular Dating by Penalised Likelihood and Maximum Likelihood</td></tr>
<tr><td style="width: 25%;"><a href="makeLabel.html">makeLabel</a></td>
<td>Label Management</td></tr>
<tr><td style="width: 25%;"><a href="makeLabel.html">makeLabel.character</a></td>
<td>Label Management</td></tr>
<tr><td style="width: 25%;"><a href="makeLabel.html">makeLabel.DNAbin</a></td>
<td>Label Management</td></tr>
<tr><td style="width: 25%;"><a href="makeLabel.html">makeLabel.multiPhylo</a></td>
<td>Label Management</td></tr>
<tr><td style="width: 25%;"><a href="makeLabel.html">makeLabel.phylo</a></td>
<td>Label Management</td></tr>
<tr><td style="width: 25%;"><a href="makeNodeLabel.html">makeNodeLabel</a></td>
<td>Makes Node Labels</td></tr>
<tr><td style="width: 25%;"><a href="mantel.test.html">mantel.test</a></td>
<td>Mantel Test for Similarity of Two Matrices</td></tr>
<tr><td style="width: 25%;"><a href="mat3.html">mat3</a></td>
<td>Three Matrices</td></tr>
<tr><td style="width: 25%;"><a href="mat5M3ID.html">mat5M3ID</a></td>
<td>Five Trees</td></tr>
<tr><td style="width: 25%;"><a href="mat5Mrand.html">mat5Mrand</a></td>
<td>Five Independent Trees</td></tr>
<tr><td style="width: 25%;"><a href="as.matching.html">matching</a></td>
<td>Conversion Between Phylo and Matching Objects</td></tr>
<tr><td style="width: 25%;"><a href="matexpo.html">matexpo</a></td>
<td>Matrix Exponential</td></tr>
<tr><td style="width: 25%;"><a href="mcconwaysims.test.html">mcconwaysims.test</a></td>
<td>McConway-Sims Test of Homogeneous Diversification</td></tr>
<tr><td style="width: 25%;"><a href="mcmc.popsize.html">mcmc.popsize</a></td>
<td>Reversible Jump MCMC to Infer Demographic History</td></tr>
<tr><td style="width: 25%;"><a href="mixedFontLabel.html">mixedFontLabel</a></td>
<td>Mixed Font Labels for Plotting</td></tr>
<tr><td style="width: 25%;"><a href="ltt.plot.html">mltt.plot</a></td>
<td>Lineages Through Time Plot</td></tr>
<tr><td style="width: 25%;"><a href="MoranI.html">Moran.I</a></td>
<td>Moran's I Autocorrelation Index</td></tr>
<tr><td style="width: 25%;"><a href="MPR.html">MPR</a></td>
<td>Most Parsimonious Reconstruction</td></tr>
<tr><td style="width: 25%;"><a href="mrca.html">mrca</a></td>
<td>Find Most Recent Common Ancestors Between Pairs</td></tr>
<tr><td style="width: 25%;"><a href="mst.html">mst</a></td>
<td>Minimum Spanning Tree</td></tr>
<tr><td style="width: 25%;"><a href="multi2di.html">multi2di</a></td>
<td>Collapse and Resolve Multichotomies</td></tr>
<tr><td style="width: 25%;"><a href="multi2di.html">multi2di.multiPhylo</a></td>
<td>Collapse and Resolve Multichotomies</td></tr>
<tr><td style="width: 25%;"><a href="multi2di.html">multi2di.phylo</a></td>
<td>Collapse and Resolve Multichotomies</td></tr>
<tr><td style="width: 25%;"><a href="multiphylo.html">multiphylo</a></td>
<td>Manipulating Lists of Trees</td></tr>
<tr><td style="width: 25%;"><a href="clustal.html">muscle</a></td>
<td>Multiple Sequence Alignment with External Applications</td></tr>
<tr><td style="width: 25%;"><a href="mvr.html">mvr</a></td>
<td>Minimum Variance Reduction</td></tr>
<tr><td style="width: 25%;"><a href="mvr.html">mvrs</a></td>
<td>Minimum Variance Reduction</td></tr>
</table>

<h2><a name="N">-- N --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="summary.phylo.html">Nedge</a></td>
<td>Print Summary of a Phylogeny</td></tr>
<tr><td style="width: 25%;"><a href="evonet.html">Nedge.evonet</a></td>
<td>Evolutionary Networks</td></tr>
<tr><td style="width: 25%;"><a href="summary.phylo.html">Nedge.multiPhylo</a></td>
<td>Print Summary of a Phylogeny</td></tr>
<tr><td style="width: 25%;"><a href="summary.phylo.html">Nedge.phylo</a></td>
<td>Print Summary of a Phylogeny</td></tr>
<tr><td style="width: 25%;"><a href="as.phylo.html">new2old.phylo</a></td>
<td>Conversion Among Tree and Network Objects</td></tr>
<tr><td style="width: 25%;"><a href="nj.html">nj</a></td>
<td>Neighbor-Joining Tree Estimation</td></tr>
<tr><td style="width: 25%;"><a href="njs.html">njs</a></td>
<td>Tree Reconstruction from Incomplete Distances With NJ* or bio-NJ*</td></tr>
<tr><td style="width: 25%;"><a href="summary.phylo.html">Nnode</a></td>
<td>Print Summary of a Phylogeny</td></tr>
<tr><td style="width: 25%;"><a href="summary.phylo.html">Nnode.multiPhylo</a></td>
<td>Print Summary of a Phylogeny</td></tr>
<tr><td style="width: 25%;"><a href="summary.phylo.html">Nnode.phylo</a></td>
<td>Print Summary of a Phylogeny</td></tr>
<tr><td style="width: 25%;"><a href="node.dating.html">node.dating</a></td>
<td>node.dating</td></tr>
<tr><td style="width: 25%;"><a href="node.depth.html">node.depth</a></td>
<td>Depth and Heights of Nodes and Tips</td></tr>
<tr><td style="width: 25%;"><a href="node.depth.html">node.depth.edgelength</a></td>
<td>Depth and Heights of Nodes and Tips</td></tr>
<tr><td style="width: 25%;"><a href="node.depth.html">node.height</a></td>
<td>Depth and Heights of Nodes and Tips</td></tr>
<tr><td style="width: 25%;"><a href="nodelabels.html">nodelabels</a></td>
<td>Labelling the Nodes, Tips, and Edges of a Tree</td></tr>
<tr><td style="width: 25%;"><a href="nodepath.html">nodepath</a></td>
<td>Find Paths of Nodes</td></tr>
<tr><td style="width: 25%;"><a href="summary.phylo.html">Ntip</a></td>
<td>Print Summary of a Phylogeny</td></tr>
<tr><td style="width: 25%;"><a href="summary.phylo.html">Ntip.multiPhylo</a></td>
<td>Print Summary of a Phylogeny</td></tr>
<tr><td style="width: 25%;"><a href="summary.phylo.html">Ntip.phylo</a></td>
<td>Print Summary of a Phylogeny</td></tr>
</table>

<h2><a name="O">-- O --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="as.phylo.html">old2new.phylo</a></td>
<td>Conversion Among Tree and Network Objects</td></tr>
</table>

<h2><a name="P">-- P --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="parafit.html">parafit</a></td>
<td>Test of host-parasite coevolution</td></tr>
<tr><td style="width: 25%;"><a href="pcoa.html">pcoa</a></td>
<td>Principal Coordinate Analysis</td></tr>
<tr><td style="width: 25%;"><a href="phydataplot.html">phydataplot</a></td>
<td>Tree Annotation</td></tr>
<tr><td style="width: 25%;"><a href="read.tree.html">phylo</a></td>
<td>Read Tree File in Parenthetic Format</td></tr>
<tr><td style="width: 25%;"><a href="phymltest.html">phymltest</a></td>
<td>Fits a Bunch of Models with PhyML</td></tr>
<tr><td style="width: 25%;"><a href="pic.html">pic</a></td>
<td>Phylogenetically Independent Contrasts</td></tr>
<tr><td style="width: 25%;"><a href="pic.ortho.html">pic.ortho</a></td>
<td>Phylogenetically Independent Orthonormal Contrasts</td></tr>
<tr><td style="width: 25%;"><a href="plot.correlogram.html">plot.correlogram</a></td>
<td>Plot a Correlogram</td></tr>
<tr><td style="width: 25%;"><a href="plot.correlogram.html">plot.correlogramList</a></td>
<td>Plot a Correlogram</td></tr>
<tr><td style="width: 25%;"><a href="evonet.html">plot.evonet</a></td>
<td>Evolutionary Networks</td></tr>
<tr><td style="width: 25%;"><a href="mst.html">plot.mst</a></td>
<td>Minimum Spanning Tree</td></tr>
<tr><td style="width: 25%;"><a href="plot.phylo.html">plot.multiPhylo</a></td>
<td>Plot Phylogenies</td></tr>
<tr><td style="width: 25%;"><a href="plot.phylo.html">plot.phylo</a></td>
<td>Plot Phylogenies</td></tr>
<tr><td style="width: 25%;"><a href="plot.phyloExtra.html">plot.phylo.extra</a></td>
<td>Extra Fuctions to Plot and Annotate Phylogenies</td></tr>
<tr><td style="width: 25%;"><a href="phymltest.html">plot.phymltest</a></td>
<td>Fits a Bunch of Models with PhyML</td></tr>
<tr><td style="width: 25%;"><a href="mcmc.popsize.html">plot.popsize</a></td>
<td>Reversible Jump MCMC to Infer Demographic History</td></tr>
<tr><td style="width: 25%;"><a href="boot.phylo.html">plot.prop.part</a></td>
<td>Tree Bipartition and Bootstrapping Phylogenies</td></tr>
<tr><td style="width: 25%;"><a href="skylineplot.html">plot.skyline</a></td>
<td>Drawing Skyline Plot Graphs</td></tr>
<tr><td style="width: 25%;"><a href="plot.varcomp.html">plot.varcomp</a></td>
<td>Plot Variance Components</td></tr>
<tr><td style="width: 25%;"><a href="plot.phyloExtra.html">plotBreakLongEdges</a></td>
<td>Extra Fuctions to Plot and Annotate Phylogenies</td></tr>
<tr><td style="width: 25%;"><a href="plotTreeTime.html">plotTreeTime</a></td>
<td>Plot Tree With Time Axis</td></tr>
<tr><td style="width: 25%;"><a href="reorder.phylo.html">postorder</a></td>
<td>Internal Reordering of Trees</td></tr>
<tr><td style="width: 25%;"><a href="compar.gee.html">predict.compar.gee</a></td>
<td>Comparative Analysis with GEEs</td></tr>
<tr><td style="width: 25%;"><a href="AAbin.html">print.AAbin</a></td>
<td>Amino Acid Sequences</td></tr>
<tr><td style="width: 25%;"><a href="ace.html">print.ace</a></td>
<td>Ancestral Character Estimation</td></tr>
<tr><td style="width: 25%;"><a href="binaryPGLMM.html">print.binaryPGLMM</a></td>
<td>Phylogenetic Generalized Linear Mixed Model for Binary Data</td></tr>
<tr><td style="width: 25%;"><a href="birthdeath.html">print.birthdeath</a></td>
<td>Estimation of Speciation and Extinction Rates With Birth-Death Models</td></tr>
<tr><td style="width: 25%;"><a href="as.bitsplits.html">print.bitsplits</a></td>
<td>Split Frequencies and Conversion Among Split Classes</td></tr>
<tr><td style="width: 25%;"><a href="chronos.html">print.chronos</a></td>
<td>Molecular Dating by Penalised Likelihood and Maximum Likelihood</td></tr>
<tr><td style="width: 25%;"><a href="compar.gee.html">print.compar.gee</a></td>
<td>Comparative Analysis with GEEs</td></tr>
<tr><td style="width: 25%;"><a href="comparePhylo.html">print.comparePhylo</a></td>
<td>Compare Two "phylo" Objects</td></tr>
<tr><td style="width: 25%;"><a href="corphylo.html">print.corphylo</a></td>
<td>Correlations among Multiple Traits with Phylogenetic Signal</td></tr>
<tr><td style="width: 25%;"><a href="DNAbin.html">print.DNAbin</a></td>
<td>Manipulate DNA Sequences in Bit-Level Format</td></tr>
<tr><td style="width: 25%;"><a href="evonet.html">print.evonet</a></td>
<td>Evolutionary Networks</td></tr>
<tr><td style="width: 25%;"><a href="lmorigin.html">print.lmorigin</a></td>
<td>Multiple regression through the origin</td></tr>
<tr><td style="width: 25%;"><a href="print.phylo.html">print.multiPhylo</a></td>
<td>Compact Display of a Phylogeny</td></tr>
<tr><td style="width: 25%;"><a href="parafit.html">print.parafit</a></td>
<td>Test of host-parasite coevolution</td></tr>
<tr><td style="width: 25%;"><a href="print.phylo.html">print.phylo</a></td>
<td>Compact Display of a Phylogeny</td></tr>
<tr><td style="width: 25%;"><a href="phymltest.html">print.phymltest</a></td>
<td>Fits a Bunch of Models with PhyML</td></tr>
<tr><td style="width: 25%;"><a href="boot.phylo.html">print.prop.part</a></td>
<td>Tree Bipartition and Bootstrapping Phylogenies</td></tr>
<tr><td style="width: 25%;"><a href="boot.phylo.html">prop.clades</a></td>
<td>Tree Bipartition and Bootstrapping Phylogenies</td></tr>
<tr><td style="width: 25%;"><a href="boot.phylo.html">prop.part</a></td>
<td>Tree Bipartition and Bootstrapping Phylogenies</td></tr>
</table>

<h2><a name="R">-- R --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="rlineage.html">rbdtree</a></td>
<td>Tree Simulation Under the Time-Dependent Birth-Death Models</td></tr>
<tr><td style="width: 25%;"><a href="DNAbin.html">rbind.DNAbin</a></td>
<td>Manipulate DNA Sequences in Bit-Level Format</td></tr>
<tr><td style="width: 25%;"><a href="rtree.html">rcoal</a></td>
<td>Generates Random Trees</td></tr>
<tr><td style="width: 25%;"><a href="rDNAbin.html">rDNAbin</a></td>
<td>Random DNA Sequences</td></tr>
<tr><td style="width: 25%;"><a href="read.caic.html">read.caic</a></td>
<td>Read Tree File in CAIC Format</td></tr>
<tr><td style="width: 25%;"><a href="read.dna.html">read.dna</a></td>
<td>Read DNA Sequences in a File</td></tr>
<tr><td style="width: 25%;"><a href="evonet.html">read.evonet</a></td>
<td>Evolutionary Networks</td></tr>
<tr><td style="width: 25%;"><a href="read.dna.html">read.FASTA</a></td>
<td>Read DNA Sequences in a File</td></tr>
<tr><td style="width: 25%;"><a href="read.dna.html">read.fastq</a></td>
<td>Read DNA Sequences in a File</td></tr>
<tr><td style="width: 25%;"><a href="read.GenBank.html">read.GenBank</a></td>
<td>Read DNA Sequences from GenBank via Internet</td></tr>
<tr><td style="width: 25%;"><a href="read.gff.html">read.gff</a></td>
<td>Read GFF Files</td></tr>
<tr><td style="width: 25%;"><a href="read.nexus.html">read.nexus</a></td>
<td>Read Tree File in Nexus Format</td></tr>
<tr><td style="width: 25%;"><a href="read.nexus.data.html">read.nexus.data</a></td>
<td>Read Character Data In NEXUS Format</td></tr>
<tr><td style="width: 25%;"><a href="read.tree.html">read.tree</a></td>
<td>Read Tree File in Parenthetic Format</td></tr>
<tr><td style="width: 25%;"><a href="reconstruct.html">reconstruct</a></td>
<td>Continuous Ancestral Character Estimation</td></tr>
<tr><td style="width: 25%;"><a href="evonet.html">reorder.evonet</a></td>
<td>Evolutionary Networks</td></tr>
<tr><td style="width: 25%;"><a href="reorder.phylo.html">reorder.multiPhylo</a></td>
<td>Internal Reordering of Trees</td></tr>
<tr><td style="width: 25%;"><a href="reorder.phylo.html">reorder.phylo</a></td>
<td>Internal Reordering of Trees</td></tr>
<tr><td style="width: 25%;"><a href="richness.yule.test.html">richness.yule.test</a></td>
<td>Test of Diversification-Shift With the Yule Process</td></tr>
<tr><td style="width: 25%;"><a href="phydataplot.html">ring</a></td>
<td>Tree Annotation</td></tr>
<tr><td style="width: 25%;"><a href="rlineage.html">rlineage</a></td>
<td>Tree Simulation Under the Time-Dependent Birth-Death Models</td></tr>
<tr><td style="width: 25%;"><a href="rtree.html">rmtree</a></td>
<td>Generates Random Trees</td></tr>
<tr><td style="width: 25%;"><a href="root.html">root</a></td>
<td>Roots Phylogenetic Trees</td></tr>
<tr><td style="width: 25%;"><a href="root.html">root.multiPhylo</a></td>
<td>Roots Phylogenetic Trees</td></tr>
<tr><td style="width: 25%;"><a href="root.html">root.phylo</a></td>
<td>Roots Phylogenetic Trees</td></tr>
<tr><td style="width: 25%;"><a href="rotate.html">rotate</a></td>
<td>Swapping Sister Clades</td></tr>
<tr><td style="width: 25%;"><a href="rotate.html">rotateConstr</a></td>
<td>Swapping Sister Clades</td></tr>
<tr><td style="width: 25%;"><a href="rlineage.html">rphylo</a></td>
<td>Tree Simulation Under the Time-Dependent Birth-Death Models</td></tr>
<tr><td style="width: 25%;"><a href="rTraitCont.html">rTraitCont</a></td>
<td>Continuous Character Simulation</td></tr>
<tr><td style="width: 25%;"><a href="rTraitDisc.html">rTraitDisc</a></td>
<td>Discrete Character Simulation</td></tr>
<tr><td style="width: 25%;"><a href="rTraitMult.html">rTraitMult</a></td>
<td>Multivariate Character Simulation</td></tr>
<tr><td style="width: 25%;"><a href="rtree.html">rtree</a></td>
<td>Generates Random Trees</td></tr>
<tr><td style="width: 25%;"><a href="rtt.html">rtt</a></td>
<td>Root a Tree by Root-to-Tip Regression</td></tr>
</table>

<h2><a name="S">-- S --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="SDM.html">SDM</a></td>
<td>Construction of Consensus Distance Matrix With SDM</td></tr>
<tr><td style="width: 25%;"><a href="seg.sites.html">seg.sites</a></td>
<td>Find Segregating Sites in DNA Sequences</td></tr>
<tr><td style="width: 25%;"><a href="skyline.html">skyline</a></td>
<td>Skyline Plot Estimate of Effective Population Size</td></tr>
<tr><td style="width: 25%;"><a href="skyline.html">skyline.coalescentIntervals</a></td>
<td>Skyline Plot Estimate of Effective Population Size</td></tr>
<tr><td style="width: 25%;"><a href="skyline.html">skyline.collapsedIntervals</a></td>
<td>Skyline Plot Estimate of Effective Population Size</td></tr>
<tr><td style="width: 25%;"><a href="skyline.html">skyline.phylo</a></td>
<td>Skyline Plot Estimate of Effective Population Size</td></tr>
<tr><td style="width: 25%;"><a href="skylineplot.html">skylineplot</a></td>
<td>Drawing Skyline Plot Graphs</td></tr>
<tr><td style="width: 25%;"><a href="skylineplot.html">skylineplot.deluxe</a></td>
<td>Drawing Skyline Plot Graphs</td></tr>
<tr><td style="width: 25%;"><a href="slowinskiguyer.test.html">slowinskiguyer.test</a></td>
<td>Slowinski-Guyer Test of Homogeneous Diversification</td></tr>
<tr><td style="width: 25%;"><a href="as.bitsplits.html">sort.bitsplits</a></td>
<td>Split Frequencies and Conversion Among Split Classes</td></tr>
<tr><td style="width: 25%;"><a href="speciesTree.html">speciesTree</a></td>
<td>Species Tree Estimation</td></tr>
<tr><td style="width: 25%;"><a href="print.phylo.html">str.multiPhylo</a></td>
<td>Compact Display of a Phylogeny</td></tr>
<tr><td style="width: 25%;"><a href="stree.html">stree</a></td>
<td>Generates Systematic Regular Trees</td></tr>
<tr><td style="width: 25%;"><a href="label2table.html">stripLabel</a></td>
<td>Label Management</td></tr>
<tr><td style="width: 25%;"><a href="subtreeplot.html">subtreeplot</a></td>
<td>Zoom on a Portion of a Phylogeny by Successive Clicks</td></tr>
<tr><td style="width: 25%;"><a href="subtrees.html">subtrees</a></td>
<td>All subtrees of a Phylogenetic Tree</td></tr>
<tr><td style="width: 25%;"><a href="summary.phylo.html">summary.phylo</a></td>
<td>Print Summary of a Phylogeny</td></tr>
<tr><td style="width: 25%;"><a href="phymltest.html">summary.phymltest</a></td>
<td>Fits a Bunch of Models with PhyML</td></tr>
<tr><td style="width: 25%;"><a href="boot.phylo.html">summary.prop.part</a></td>
<td>Tree Bipartition and Bootstrapping Phylogenies</td></tr>
</table>

<h2><a name="T">-- T --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="clustal.html">tcoffee</a></td>
<td>Multiple Sequence Alignment with External Applications</td></tr>
<tr><td style="width: 25%;"><a href="nodelabels.html">tiplabels</a></td>
<td>Labelling the Nodes, Tips, and Edges of a Tree</td></tr>
<tr><td style="width: 25%;"><a href="trans.html">trans</a></td>
<td>Translation from DNA to Amino Acid Sequences</td></tr>
<tr><td style="width: 25%;"><a href="treePop.html">treePop</a></td>
<td>Tree Popping</td></tr>
<tr><td style="width: 25%;"><a href="trex.html">trex</a></td>
<td>Tree Explorer With Multiple Devices</td></tr>
<tr><td style="width: 25%;"><a href="triangMtd.html">triangMtd</a></td>
<td>Tree Reconstruction Based on the Triangles Method</td></tr>
<tr><td style="width: 25%;"><a href="triangMtd.html">triangMtds</a></td>
<td>Tree Reconstruction Based on the Triangles Method</td></tr>
</table>

<h2><a name="U">-- U --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="additive.html">ultrametric</a></td>
<td>Incomplete Distance Matrix Filling</td></tr>
<tr><td style="width: 25%;"><a href="unique.multiPhylo.html">unique.multiPhylo</a></td>
<td>Revomes Duplicate Trees</td></tr>
<tr><td style="width: 25%;"><a href="root.html">unroot</a></td>
<td>Roots Phylogenetic Trees</td></tr>
<tr><td style="width: 25%;"><a href="root.html">unroot.multiPhylo</a></td>
<td>Roots Phylogenetic Trees</td></tr>
<tr><td style="width: 25%;"><a href="root.html">unroot.phylo</a></td>
<td>Roots Phylogenetic Trees</td></tr>
<tr><td style="width: 25%;"><a href="updateLabel.html">updateLabel</a></td>
<td>Update Labels</td></tr>
<tr><td style="width: 25%;"><a href="updateLabel.html">updateLabel.AAbin</a></td>
<td>Update Labels</td></tr>
<tr><td style="width: 25%;"><a href="updateLabel.html">updateLabel.character</a></td>
<td>Update Labels</td></tr>
<tr><td style="width: 25%;"><a href="updateLabel.html">updateLabel.data.frame</a></td>
<td>Update Labels</td></tr>
<tr><td style="width: 25%;"><a href="updateLabel.html">updateLabel.DNAbin</a></td>
<td>Update Labels</td></tr>
<tr><td style="width: 25%;"><a href="updateLabel.html">updateLabel.evonet</a></td>
<td>Update Labels</td></tr>
<tr><td style="width: 25%;"><a href="updateLabel.html">updateLabel.matrix</a></td>
<td>Update Labels</td></tr>
<tr><td style="width: 25%;"><a href="updateLabel.html">updateLabel.phylo</a></td>
<td>Update Labels</td></tr>
</table>

<h2><a name="V">-- V --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="varcomp.html">varcomp</a></td>
<td>Compute Variance Component Estimates</td></tr>
<tr><td style="width: 25%;"><a href="varCompPhylip.html">varCompPhylip</a></td>
<td>Variance Components with Orthonormal Contrasts</td></tr>
<tr><td style="width: 25%;"><a href="vcv.phylo.html">vcv</a></td>
<td>Phylogenetic Variance-covariance or Correlation Matrix</td></tr>
<tr><td style="width: 25%;"><a href="vcv.phylo.html">vcv.corPhyl</a></td>
<td>Phylogenetic Variance-covariance or Correlation Matrix</td></tr>
<tr><td style="width: 25%;"><a href="vcv.phylo.html">vcv.phylo</a></td>
<td>Phylogenetic Variance-covariance or Correlation Matrix</td></tr>
<tr><td style="width: 25%;"><a href="vcv2phylo.html">vcv2phylo</a></td>
<td>Variance-Covariance Matrix to Tree</td></tr>
</table>

<h2><a name="W">-- W --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="weight.taxo.html">weight.taxo</a></td>
<td>Define Similarity Matrix</td></tr>
<tr><td style="width: 25%;"><a href="weight.taxo.html">weight.taxo2</a></td>
<td>Define Similarity Matrix</td></tr>
<tr><td style="width: 25%;"><a href="where.html">where</a></td>
<td>Find Patterns in DNA Sequences</td></tr>
<tr><td style="width: 25%;"><a href="which.edge.html">which.edge</a></td>
<td>Identifies Edges of a Tree</td></tr>
<tr><td style="width: 25%;"><a href="woodmouse.html">woodmouse</a></td>
<td>Cytochrome b Gene Sequences of Woodmice</td></tr>
<tr><td style="width: 25%;"><a href="write.dna.html">write.dna</a></td>
<td>Write DNA Sequences in a File</td></tr>
<tr><td style="width: 25%;"><a href="evonet.html">write.evonet</a></td>
<td>Evolutionary Networks</td></tr>
<tr><td style="width: 25%;"><a href="write.dna.html">write.FASTA</a></td>
<td>Write DNA Sequences in a File</td></tr>
<tr><td style="width: 25%;"><a href="write.nexus.html">write.nexus</a></td>
<td>Write Tree File in Nexus Format</td></tr>
<tr><td style="width: 25%;"><a href="write.nexus.data.html">write.nexus.data</a></td>
<td>Write Character Data in NEXUS Format</td></tr>
<tr><td style="width: 25%;"><a href="write.tree.html">write.tree</a></td>
<td>Write Tree File in Parenthetic Format</td></tr>
</table>

<h2><a name="X">-- X --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="apetools.html">Xplor</a></td>
<td>Tools to Explore Files</td></tr>
<tr><td style="width: 25%;"><a href="apetools.html">Xplorefiles</a></td>
<td>Tools to Explore Files</td></tr>
</table>

<h2><a name="Y">-- Y --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="yule.html">yule</a></td>
<td>Fits the Yule Model to a Phylogenetic Tree</td></tr>
<tr><td style="width: 25%;"><a href="yule.cov.html">yule.cov</a></td>
<td>Fits the Yule Model With Covariates</td></tr>
<tr><td style="width: 25%;"><a href="yule.time.html">yule.time</a></td>
<td>Fits the Time-Dependent Yule Model</td></tr>
</table>

<h2><a name="Z">-- Z --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="zoom.html">zoom</a></td>
<td>Zoom on a Portion of a Phylogeny</td></tr>
</table>

<h2><a name="misc">-- misc --</a></h2>

<table width="100%">
<tr><td style="width: 25%;"><a href="multiphylo.html">$.multiPhylo</a></td>
<td>Manipulating Lists of Trees</td></tr>
<tr><td style="width: 25%;"><a href="multiphylo.html">$&lt;-.multiPhylo</a></td>
<td>Manipulating Lists of Trees</td></tr>
<tr><td style="width: 25%;"><a href="bind.tree.html">+.phylo</a></td>
<td>Binds Trees</td></tr>
<tr><td style="width: 25%;"><a href="c.phylo.html">.compressTipLabel</a></td>
<td>Building Lists of Trees</td></tr>
<tr><td style="width: 25%;"><a href="c.phylo.html">.uncompressTipLabel</a></td>
<td>Building Lists of Trees</td></tr>
<tr><td style="width: 25%;"><a href="AAbin.html">[.AAbin</a></td>
<td>Amino Acid Sequences</td></tr>
<tr><td style="width: 25%;"><a href="DNAbin.html">[.DNAbin</a></td>
<td>Manipulate DNA Sequences in Bit-Level Format</td></tr>
<tr><td style="width: 25%;"><a href="multiphylo.html">[.multiPhylo</a></td>
<td>Manipulating Lists of Trees</td></tr>
<tr><td style="width: 25%;"><a href="multiphylo.html">[&lt;-.multiPhylo</a></td>
<td>Manipulating Lists of Trees</td></tr>
<tr><td style="width: 25%;"><a href="multiphylo.html">[[.multiPhylo</a></td>
<td>Manipulating Lists of Trees</td></tr>
<tr><td style="width: 25%;"><a href="multiphylo.html">[[&lt;-.multiPhylo</a></td>
<td>Manipulating Lists of Trees</td></tr>
</table>
</body></html>
