<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html xmlns="http://www.w3.org/1999/xhtml"><head><title>R: Read a mask from a file</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<link rel="stylesheet" type="text/css" href="R.css" />
</head><body>

<table width="100%" summary="page for read.Mask {IRanges}"><tr><td>read.Mask {IRanges}</td><td style="text-align: right;">R Documentation</td></tr></table>

<h2>Read a mask from a file</h2>

<h3>Description</h3>

<p><code>read.agpMask</code> and <code>read.gapMask</code> extract the AGAPS mask from an
NCBI &quot;agp&quot; file or a UCSC &quot;gap&quot; file, respectively.
</p>
<p><code>read.liftMask</code> extracts the AGAPS mask from a UCSC &quot;lift&quot; file
(i.e. a file containing offsets of contigs within sequences).
</p>
<p><code>read.rmMask</code> extracts the RM mask from a RepeatMasker .out file.
</p>
<p><code>read.trfMask</code> extracts the TRF mask from a Tandem Repeats Finder .bed
file.
</p>


<h3>Usage</h3>

<pre>
read.agpMask(file, seqname="?", mask.width=NA, gap.types=NULL, use.gap.types=FALSE)
read.gapMask(file, seqname="?", mask.width=NA, gap.types=NULL, use.gap.types=FALSE)
read.liftMask(file, seqname="?", mask.width=NA)
read.rmMask(file, seqname="?", mask.width=NA, use.IDs=FALSE)
read.trfMask(file, seqname="?", mask.width=NA)
</pre>


<h3>Arguments</h3>

<table summary="R argblock">
<tr valign="top"><td><code>file</code></td>
<td>

<p>Either a character string naming a file or a connection open
for reading.
</p>
</td></tr>
<tr valign="top"><td><code>seqname</code></td>
<td>

<p>The name of the sequence for which the mask must be extracted.
If no sequence is specified (i.e. <code>seqname="?"</code>) then an error is
raised and the sequence names found in the file are displayed.
If the file doesn't contain any information for the specified sequence,
then a warning is issued and an empty mask of width <code>mask.width</code>
is returned.
</p>
</td></tr>
<tr valign="top"><td><code>mask.width</code></td>
<td>

<p>The width of the mask to return i.e. the length of the sequence this
mask will be put on.
See <code>?`<a href="MaskCollection-class.html">MaskCollection-class</a>`</code> for more information about
the width of a <a href="MaskCollection-class.html">MaskCollection</a> object.
</p>
</td></tr>
<tr valign="top"><td><code>gap.types</code></td>
<td>

<p><code>NULL</code> or a character vector containing gap types.
Use this argument to filter the assembly gaps that are to be extracted
from the &quot;agp&quot; or &quot;gap&quot; file based on their type. Most common gap types
are <code>"contig"</code>, <code>"clone"</code>, <code>"centromere"</code>, <code>"telomere"</code>,
<code>"heterochromatin"</code>, <code>"short_arm"</code> and <code>"fragment"</code>.
With <code>gap.types=NULL</code>, all the assembly gaps described in the file
are extracted.
With <code>gap.types="?"</code>, an error is raised and the gap types found
in the file for the specified sequence are displayed.
</p>
</td></tr>
<tr valign="top"><td><code>use.gap.types</code></td>
<td>

<p>Whether or not the gap types provided in the &quot;agp&quot; or &quot;gap&quot; file should
be used to name the ranges constituing the returned mask.
See <code>?`<a href="IRanges-class.html">IRanges-class</a>`</code> for more information about the
names of an <a href="IRanges-constructor.html">IRanges</a> object.
</p>
</td></tr>
<tr valign="top"><td><code>use.IDs</code></td>
<td>

<p>Whether or not the repeat IDs provided in the RepeatMasker .out file
should be used to name the ranges constituing the returned mask.
See <code>?`<a href="IRanges-class.html">IRanges-class</a>`</code> for more information about the
names of an <a href="IRanges-constructor.html">IRanges</a> object.
</p>
</td></tr>
</table>


<h3>See Also</h3>

<p><a href="MaskCollection-class.html">MaskCollection-class</a>,
<a href="IRanges-class.html">IRanges-class</a>
</p>


<h3>Examples</h3>

<pre>
  ## ---------------------------------------------------------------------
  ## A. Extract a mask of assembly gaps ("AGAPS" mask) with read.agpMask()
  ## ---------------------------------------------------------------------
  ## Note: The hs_b36v3_chrY.agp file was obtained by downloading,
  ## extracting and renaming the hs_ref_chrY.agp.gz file from
  ##
  ##   ftp://ftp.ncbi.nih.gov/genomes/H_sapiens/Assembled_chromosomes/
  ##     hs_ref_chrY.agp.gz      5 KB  24/03/08  04:33:00 PM
  ##
  ## on May 9, 2008.

  chrY_length &lt;- 57772954
  file1 &lt;- system.file("extdata", "hs_b36v3_chrY.agp", package="IRanges")
  mask1 &lt;- read.agpMask(file1, seqname="chrY", mask.width=chrY_length,
                        use.gap.types=TRUE)
  mask1
  mask1[[1]]

  mask11 &lt;- read.agpMask(file1, seqname="chrY", mask.width=chrY_length,
                         gap.types=c("centromere", "heterochromatin"))
  mask11[[1]]

  ## ---------------------------------------------------------------------
  ## B. Extract a mask of assembly gaps ("AGAPS" mask) with read.liftMask()
  ## ---------------------------------------------------------------------
  ## Note: The hg18liftAll.lft file was obtained by downloading,
  ## extracting and renaming the liftAll.zip file from
  ##
  ##   http://hgdownload.cse.ucsc.edu/goldenPath/hg18/bigZips/
  ##     liftAll.zip             03-Feb-2006 11:35  5.5K
  ##
  ## on May 8, 2008.

  file2 &lt;- system.file("extdata", "hg18liftAll.lft", package="IRanges")
  mask2 &lt;- read.liftMask(file2, seqname="chr1")
  mask2
  if (interactive()) {
      ## contigs 7 and 8 for chrY are adjacent
      read.liftMask(file2, seqname="chrY")

      ## displays the sequence names found in the file
      read.liftMask(file2)

      ## specify an unknown sequence name
      read.liftMask(file2, seqname="chrZ", mask.width=300)
  }

  ## ---------------------------------------------------------------------
  ## C. Extract a RepeatMasker ("RM") or Tandem Repeats Finder ("TRF")
  ##    mask with read.rmMask() or read.trfMask()
  ## ---------------------------------------------------------------------
  ## Note: The ce2chrM.fa.out and ce2chrM.bed files were obtained by
  ## downloading, extracting and renaming the chromOut.zip and
  ## chromTrf.zip files from
  ##
  ##   http://hgdownload.cse.ucsc.edu/goldenPath/ce2/bigZips/
  ##     chromOut.zip            21-Apr-2004 09:05  2.6M
  ##     chromTrf.zip            21-Apr-2004 09:07  182K
  ##
  ## on May 7, 2008.

  ## Before you can extract a mask with read.rmMask() or read.trfMask(), you
  ## need to know the length of the sequence that you're going to put the
  ## mask on:
  if (interactive()) {
      library(BSgenome.Celegans.UCSC.ce2)
      chrM_length &lt;- seqlengths(Celegans)[["chrM"]]

      ## Read the RepeatMasker .out file for chrM in ce2:
      file3 &lt;- system.file("extdata", "ce2chrM.fa.out", package="IRanges")
      RMmask &lt;- read.rmMask(file3, seqname="chrM", mask.width=chrM_length)
      RMmask

      ## Read the Tandem Repeats Finder .bed file for chrM in ce2:
      file4 &lt;- system.file("extdata", "ce2chrM.bed", package="IRanges")
      TRFmask &lt;- read.trfMask(file4, seqname="chrM", mask.width=chrM_length)
      TRFmask
      desc(TRFmask) &lt;- paste(desc(TRFmask), "[period&lt;=12]")
      TRFmask

      ## Put the 2 masks on chrM:
      chrM &lt;- Celegans$chrM
      masks(chrM) &lt;- RMmask  # this would drop all current masks, if any
      masks(chrM) &lt;- append(masks(chrM), TRFmask)
      chrM
  }
</pre>

<hr /><div style="text-align: center;">[Package <em>IRanges</em> version 2.16.0 <a href="00Index.html">Index</a>]</div>
</body></html>
